2014-07-10 19:10:55 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-10 19:10:57 get_access_token.py[line:34] INFO access_token:e83NqovhmCTwksj8VJmN9ZqNMjdPHxHSyLeqn4S_20Ecv0g0UM5U2ri2ImnCUOyeAb_bKdZ3m-DrX4k_iK2TlA
2014-07-10 19:11:13 web.py[line:1728] INFO 200 GET /bind/99999 (127.0.0.1) 4.40ms
2014-07-10 19:11:23 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-10 19:11:24 web.py[line:1728] INFO 200 POST /bind/99999 (127.0.0.1) 1013.10ms
2014-07-10 19:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.52ms
2014-07-10 19:33:41 web.py[line:1728] INFO 200 POST /wechat?signature=6f6cc54ae1c47b86dc70e230797f39010e223d90&timestamp=1404992019&nonce=1486909145 (127.0.0.1) 2.08ms
2014-07-10 19:34:05 web.py[line:1728] INFO 200 POST /wechat?signature=f393b14c918408c62a1a8983c6f2c8a731e4786b&timestamp=1404992043&nonce=1911089807 (127.0.0.1) 4.17ms
2014-07-10 19:34:49 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.94ms
2014-07-10 19:35:21 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-10 19:35:25 web.py[line:1728] INFO 200 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 3166.28ms
2014-07-10 19:35:43 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=16b2ecab28c0793462725e7b83c0e8da97b7d902&timestamp=1404992143&nonce=1625681087 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=16b2ecab28c0793462725e7b83c0e8da97b7d902&timestamp=1404992143&nonce=1625681087', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
NameError: global name 'send' is not defined
2014-07-10 19:35:43 web.py[line:1728] ERROR 500 POST /wechat?signature=16b2ecab28c0793462725e7b83c0e8da97b7d902&timestamp=1404992143&nonce=1625681087 (127.0.0.1) 4.05ms
2014-07-10 20:10:57 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-10 20:10:59 get_access_token.py[line:34] INFO access_token:jHwShj8jORXxfYJOIIiEtku_ERnWvUlUWkwo5qdbPymUcbdWjTx_xzfWsCpzjKDqzAP1DXQCekppWZQY2B7oCw
2014-07-10 20:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-10 21:10:59 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-10 21:11:00 get_access_token.py[line:34] INFO access_token:F9VsSWKs25cUHI6Imq9fG4psZ0qxeUYu8Lp50goDKWoAKCGfPkyw3hQ2cSLwkkU0BCBOjm7hIjhS_m9OPwhq5A
2014-07-10 21:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-10 22:11:00 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-10 22:11:02 get_access_token.py[line:34] INFO access_token:05bd_hr2rwAkigz1RsPcXhEMK3U0ZTu6UAWT6JZ6iAqYwFQ89ovDYE8ZrS6ATEhRqojWfuXS86PrKzn4-kKWfA
2014-07-10 22:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-10 23:11:02 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-10 23:11:03 get_access_token.py[line:34] INFO access_token:QtzUHi4yOzZURRYTNtAAx5gmEsI3IUp_QITCIMA00E6fprrO_GbXi-tMXEaLnUcZ8Khoa8VM6XCKE_Ng1rrfXw
2014-07-10 23:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-10 23:54:30 web.py[line:1728] INFO 200 POST /wechat?signature=cdd54336f16e13b7d3d959cc0fe8a85c40c31dd2&timestamp=1405007668&nonce=1258021066 (127.0.0.1) 1.12ms
2014-07-11 00:11:03 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 00:11:05 get_access_token.py[line:34] INFO access_token:TSMYs6I_9VJKHE2T-LMikHpHxES8VoY_pg67ZdKHERjwxI2gzV6uwpo0L60NtffGXATW8xMXmOW7Y9iw0qhXYQ
2014-07-11 00:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.54ms
2014-07-11 01:11:05 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 01:11:07 get_access_token.py[line:34] INFO access_token:QrfximSHdd420_lVHF4fZhCq3xHqPuqoLHrxqizr9jO3t6XRlDv5WJQn-4IdPLbGpTWs-9CuTrJmaqaCZrZCXQ
2014-07-11 01:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-11 02:11:07 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 02:11:08 get_access_token.py[line:34] INFO access_token:a4yny7jGh1P8j0kzDYU2ekVkORCrblPPf22AdMQYrecxbJNQtOsj7DXzLIYnzMLyXLOiohP99FT-3UlslLE0aw
2014-07-11 02:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.54ms
2014-07-11 03:11:08 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 03:11:10 get_access_token.py[line:34] INFO access_token:2kOCSMHV0ZJE9jUmzjjOa5EJp59Kw44Cfk1YGA-wJQvMe1TFfhUs9YFphygm0lY_zrDMyEz_XeksCRHLVSVnpA
2014-07-11 03:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.45ms
2014-07-11 04:11:10 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 04:11:11 get_access_token.py[line:34] INFO access_token:QpEG7VnolcdIGOp8tZWW2DXIq4EQ8cx3r54iqnKSV2ENElHYApBBM9UUfIOizFcK_-KM32-VDMKVJJc82zaONw
2014-07-11 04:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.47ms
2014-07-11 05:11:11 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 05:11:13 get_access_token.py[line:34] INFO access_token:TjNiLRQ60OFKxoaymQ8QZ-QSEPrC__dZg6DodqrwhnmrA42oiPcpDCTqgK7hOZhI9XNUUzBkrG2MII0yCWKlTw
2014-07-11 05:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.49ms
2014-07-11 06:11:13 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 06:11:15 get_access_token.py[line:34] INFO access_token:a4yny7jGh1P8j0kzDYU2ehemBEodgCCNNUzDto7xTJQMdcsJ5w-WIPWhQeYhIbzp2BtLp6Fgb8TljWCcUbw2iw
2014-07-11 06:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-11 07:11:15 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 07:11:17 get_access_token.py[line:34] INFO access_token:JF_cl8yTbN3tzCSfC032D5dvNSS1rnatEkR-HXnrqWyOGZCpHqRchJS8rcXIhO8C7psP7S9i1MpB30cLuUPy_Q
2014-07-11 07:27:07 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-11 08:11:17 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 08:11:18 get_access_token.py[line:34] INFO access_token:0N45cNfDNDwpVIJU21AwIdDK7sSyLXHu2uwtnm5BtylRmrz5og67HzIpyPh7T36gLQbUiBPV2DZyQV0EC9QF8Q
2014-07-11 08:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-11 09:11:18 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 09:11:20 get_access_token.py[line:34] INFO access_token:GazAHkDiviG65sW4vbg4L-EMKgmKZVSr2VWBbV_U7qq2ZY68A453u8F_Ez3kukX6PK9NH_XP9RVZiXmlwo1fbg
2014-07-11 09:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-11 10:11:20 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 10:11:21 get_access_token.py[line:34] INFO access_token:hU9tkLyC-ifE9cnVr-zv8WPs0aBXojkz3oK-1Eud-oRmTwHFJrKeK02rKZUWTiWVdH16OTprW3bDWOtc342XmA
2014-07-11 10:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.41ms
2014-07-11 11:11:21 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 11:11:23 get_access_token.py[line:34] INFO access_token:99j3VhFKeQzqj6zNJy0RSQWZ_edv8jnHjK4T9gsdsq_uQqbzMRCu2JQknN-4AX-7bx7lhs5vdRu9ocjD31zn4A
2014-07-11 11:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-11 12:11:23 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 12:11:24 get_access_token.py[line:34] INFO access_token:nPK6hRz4jjEGKZ-e_oKpimI6DYTdJwwkQT8WRoX0wK70KWrgiCf9KryrpPc68aw_t4Y2T9UCZ8aZSXACb9L9rw
2014-07-11 12:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.41ms
2014-07-11 12:30:56 web.py[line:1728] WARNING 404 GET /bind (127.0.0.1) 0.40ms
2014-07-11 12:31:39 web.py[line:1728] WARNING 404 GET /bind (127.0.0.1) 0.46ms
2014-07-11 13:11:24 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 13:11:26 get_access_token.py[line:34] INFO access_token:QAP-Csx-tGUjnkraTOWPgXje45Sagcw9KSBYrA6JbS8qdqKdsInhlB93zb03bnN-QQnAUkB7B2BKItZx3ckT5Q
2014-07-11 13:19:06 web.py[line:1728] INFO 200 POST /wechat?signature=2ce1af2eb8ff8ed9126141ba56b5134353333282&timestamp=1405055943&nonce=1100817173 (127.0.0.1) 1.22ms
2014-07-11 13:19:24 web.py[line:1728] INFO 200 GET /bind/oDVlVuIMhughDn6jWUOB2GcsU2w8 (127.0.0.1) 0.63ms
2014-07-11 13:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.45ms
2014-07-11 13:39:11 web.py[line:1728] WARNING 404 GET /bind (127.0.0.1) 0.41ms
2014-07-11 13:49:07 web.py[line:1728] INFO 200 GET /bind/1000 (127.0.0.1) 0.59ms
2014-07-11 13:49:07 web.py[line:1728] INFO 200 GET /bind/1000 (127.0.0.1) 0.50ms
2014-07-11 14:11:26 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 14:11:28 get_access_token.py[line:34] INFO access_token:5ccWMym-x7A5aVZc9gC7MYHu48F6yb1IFXTvA9q94pAvcjx0m2L4uCRcEfnjmqgSzicZmjwxiFbC6uNU0Pv5IQ
2014-07-11 14:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-11 15:11:28 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 15:11:29 get_access_token.py[line:34] INFO access_token:GGsgJHcnKIVqQLWKr6MsIicUPN-Fj0RXbkPwCSgE42dM_vmKcSf4RsIBO_iyhP_8Sh6qckAsLaZm2F945SY-Qg
2014-07-11 15:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-11 16:11:30 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 16:11:31 get_access_token.py[line:34] INFO access_token:e83NqovhmCTwksj8VJmN9aCY0man71d7t-2VD7vkxwR2yGFT8m-bGq0ifOSIHFqf2UtKn13kV6oDal4TJuNUIQ
2014-07-11 16:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-11 17:11:31 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 17:11:33 get_access_token.py[line:34] INFO access_token:VWrenSRffQstEQL-n64aNKZU2P-ea6saSJphKPCV6Vz7XHRaJj6dl6whzHnhaMOIgKRUJFIdAoO0LjrJSbpt6g
2014-07-11 17:23:49 web.py[line:1728] INFO 200 POST /wechat?signature=a930e7a2f84ede4a65685808d9e59c62057c7b1f&timestamp=1405070624&nonce=1524054858 (127.0.0.1) 0.88ms
2014-07-11 17:23:59 web.py[line:1728] INFO 200 POST /wechat?signature=ca59423622e4fe6937656895d54458bbf0390f47&timestamp=1405070639&nonce=1299622883 (127.0.0.1) 1.20ms
2014-07-11 17:24:08 web.py[line:1728] INFO 200 POST /wechat?signature=a9fc37ba91cb1ad40479d92b348ae42dd3eb800b&timestamp=1405070648&nonce=274041254 (127.0.0.1) 0.89ms
2014-07-11 17:26:19 web.py[line:1305] ERROR Uncaught exception GET /tail/1111 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='GET', uri='/tail/1111', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Connection': 'close', 'Accept-Language': 'en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/tail.py", line 12, in get
    self.render('login.html', action=action)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 615, in render
    html = self.render_string(template_name, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 722, in render_string
    return t.generate(**namespace)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/template.py", line 278, in generate
    return execute()
  File "login_html.generated.py", line 5, in _tt_execute
    _tt_tmp = info  # login.html:5
NameError: global name 'info' is not defined
2014-07-11 17:26:19 web.py[line:1728] ERROR 500 GET /tail/1111 (127.0.0.1) 2.29ms
2014-07-11 17:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.53ms
2014-07-11 17:35:42 web.py[line:1305] ERROR Uncaught exception GET /tail/1111 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='GET', uri='/tail/1111', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Connection': 'close', 'Accept-Language': 'en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Cache-Control': 'max-age=0', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/tail.py", line 15, in get
    self.render('login.html', action=action)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 615, in render
    html = self.render_string(template_name, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 722, in render_string
    return t.generate(**namespace)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/template.py", line 278, in generate
    return execute()
  File "login_html.generated.py", line 5, in _tt_execute
    _tt_tmp = info  # login.html:5
NameError: global name 'info' is not defined
2014-07-11 17:35:42 web.py[line:1728] ERROR 500 GET /tail/1111 (127.0.0.1) 6.48ms
2014-07-11 17:38:13 web.py[line:1728] INFO 200 GET /tail/1111 (127.0.0.1) 5.84ms
2014-07-11 17:38:59 web.py[line:1305] ERROR Uncaught exception POST /tail/1111 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/tail/1111', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Origin': 'http://bandwagon.shijunzi.info', 'Content-Length': '9', 'Accept-Language': 'en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36', 'Connection': 'close', 'Referer': 'http://bandwagon.shijunzi.info/elfin/tail/1111', 'Cache-Control': 'max-age=0', 'Content-Type': 'application/x-www-form-urlencoded'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/tail.py", line 20, in post
    send(weixin_id,tail=tail)
NameError: global name 'send' is not defined
2014-07-11 17:38:59 web.py[line:1728] ERROR 500 POST /tail/1111 (127.0.0.1) 2.15ms
2014-07-11 18:11:33 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 18:11:35 get_access_token.py[line:34] INFO access_token:3_TUSnjYKug0WjUYTqLnEgpo0PmRtF5fV8wJdpNs-Jrgox69pHoIlyi9Ft329RWFhKSBW3vrwwSmJNZZJa4GzA
2014-07-11 18:23:30 web.py[line:1728] INFO 200 POST /wechat?signature=9fbefa0af13c89da732c815011de9c953e8cda6f&timestamp=1405074209&nonce=855500421 (127.0.0.1) 4.88ms
2014-07-11 18:23:51 web.py[line:1728] INFO 200 POST /wechat?signature=e8aa1476d1a58dc2b08cff1cc331163def01e5fd&timestamp=1405074230&nonce=220047198 (127.0.0.1) 3.82ms
2014-07-11 18:25:17 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 2.24ms
2014-07-11 18:26:38 web.py[line:1728] INFO 200 GET /bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg (127.0.0.1) 0.56ms
2014-07-11 18:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.40ms
2014-07-11 18:27:08 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.47ms
2014-07-11 18:43:50 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=ced8f1e55f22d8c6aea057fe764f1cee000c56ae&timestamp=1405075429&nonce=964451332 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=ced8f1e55f22d8c6aea057fe764f1cee000c56ae&timestamp=1405075429&nonce=964451332', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '341', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
NameError: global name 'send' is not defined
2014-07-11 18:43:50 web.py[line:1728] ERROR 500 POST /wechat?signature=ced8f1e55f22d8c6aea057fe764f1cee000c56ae&timestamp=1405075429&nonce=964451332 (127.0.0.1) 2.76ms
2014-07-11 19:01:42 web.py[line:1728] INFO 304 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.69ms
2014-07-11 19:11:35 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 19:11:37 get_access_token.py[line:34] INFO access_token:aYKMyfY_Uzn2U0G94DUmUIvblVCaH2fI7rYcf8j6vXpRukIZYN8wdOQTlmCB8e-vX1_ocwQpaT8_tBL9O2uqxA
2014-07-11 19:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.43ms
2014-07-11 19:34:17 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 5.65ms
2014-07-11 19:34:17 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/logo.gif (127.0.0.1) 0.84ms
2014-07-11 19:34:18 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 4.11ms
2014-07-11 19:35:23 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.20ms
2014-07-11 19:35:24 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/logo.gif (127.0.0.1) 0.85ms
2014-07-11 19:35:24 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 1.20ms
2014-07-11 19:58:26 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 4.76ms
2014-07-11 19:58:27 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.86ms
2014-07-11 20:11:37 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 20:11:39 get_access_token.py[line:34] INFO access_token:QtzUHi4yOzZURRYTNtAAxxiVI4e9evayA_O4MroqJ8dYTmDh1Ffz9EkrOOT2h1A9Mdi0R4XbdmBnzMjWmqobDQ
2014-07-11 20:13:59 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 4.81ms
2014-07-11 20:14:00 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.90ms
2014-07-11 20:14:29 web.py[line:1728] INFO 200 GET /bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg (127.0.0.1) 3.77ms
2014-07-11 20:14:30 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.90ms
2014-07-11 20:15:49 web.py[line:1728] INFO 200 GET /bind/static/logo.gif (127.0.0.1) 1.23ms
2014-07-11 20:15:49 web.py[line:1728] INFO 200 GET /bind/static/xiezhua_files/ga.php (127.0.0.1) 0.99ms
2014-07-11 20:16:02 web.py[line:1728] INFO 200 GET /bind/static/logo.gif (127.0.0.1) 0.88ms
2014-07-11 20:16:03 web.py[line:1728] INFO 200 GET /bind/static/xiezhua_files/ga.php (127.0.0.1) 0.91ms
2014-07-11 20:16:13 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 5.35ms
2014-07-11 20:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.48ms
2014-07-11 20:52:13 web.py[line:1728] INFO 200 GET /bind/ssdfgsdfgsdfg (127.0.0.1) 0.71ms
2014-07-11 20:52:13 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.67ms
2014-07-11 20:52:45 web.py[line:1728] INFO 200 GET /bind/ssdfgsdfgsdfg (127.0.0.1) 4.75ms
2014-07-11 20:52:45 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.93ms
2014-07-11 20:53:11 web.py[line:1728] INFO 200 GET /bind/statisdfgsdfgsdfg (127.0.0.1) 2.10ms
2014-07-11 20:53:11 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.87ms
2014-07-11 20:56:40 web.py[line:1728] INFO 200 GET /bind/statisdfgsdfgsdfg (127.0.0.1) 7.85ms
2014-07-11 20:56:40 web.py[line:1728] INFO 200 GET /bind/static_url('logo.gif') (127.0.0.1) 0.88ms
2014-07-11 20:56:41 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 2.66ms
2014-07-11 21:00:12 web.py[line:1728] INFO 200 GET /bind/statisdfgsdfgsdfg (127.0.0.1) 5.89ms
2014-07-11 21:00:12 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.97ms
2014-07-11 21:00:37 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 5.45ms
2014-07-11 21:00:48 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.70ms
2014-07-11 21:11:39 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 21:11:40 get_access_token.py[line:34] INFO access_token:EhokNy96sCG38d0y2ma7AHsqW53DzBJwz4w-eTHWALD2r31XZxt7IAgKkno5S1aXCwifmj9WvVsqG0lxJoSrzA
2014-07-11 21:13:22 web.py[line:1728] INFO 200 GET /bind/statisdfgsdfgsdfg (127.0.0.1) 5.98ms
2014-07-11 21:13:22 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/logo.gif (127.0.0.1) 0.93ms
2014-07-11 21:13:22 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 2.59ms
2014-07-11 21:13:44 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/logo.gif (127.0.0.1) 0.87ms
2014-07-11 21:13:44 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/http/bandwagon.shijunzi.info/elfin/static/logo.gif (127.0.0.1) 1.06ms
2014-07-11 21:13:44 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/xiezhua_files/ga.php (127.0.0.1) 0.67ms
2014-07-11 21:13:50 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/logo.gif (127.0.0.1) 1.12ms
2014-07-11 21:13:50 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/http/bandwagon.shijunzi.info/elfin/static/logo.gif (127.0.0.1) 1.01ms
2014-07-11 21:13:50 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/xiezhua_files/ga.php (127.0.0.1) 0.69ms
2014-07-11 21:26:49 web.py[line:1728] INFO 200 GET /bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg (127.0.0.1) 4.59ms
2014-07-11 21:26:49 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.87ms
2014-07-11 21:26:49 web.py[line:1728] INFO 200 GET /bind/http/bandwagon.shijunzi.info/elfin/static/logo.gif (127.0.0.1) 3.66ms
2014-07-11 21:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.51ms
2014-07-11 21:29:21 web.py[line:1305] ERROR Uncaught exception GET /bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='GET', uri='/bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Accept-Language': 'en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36', 'Connection': 'close', 'Cache-Control': 'max-age=0', 'If-None-Match': '"cde88b36bb61e2830e3943a62325dfea08620fd7"'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 15, in get
    self.render('login.html', time=time.ctime(), action=action)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 615, in render
    html = self.render_string(template_name, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 722, in render_string
    return t.generate(**namespace)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/template.py", line 278, in generate
    return execute()
  File "login_html.generated.py", line 5, in _tt_execute
    _tt_tmp = static_url("logo.gif")  # login.html:12
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1096, in static_url
    self.require_setting("static_path", "static_url")
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1135, in require_setting
    "application to use %s" % (name, feature))
Exception: You must define the 'static_path' setting in your application to use static_url
2014-07-11 21:29:21 web.py[line:1728] ERROR 500 GET /bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg (127.0.0.1) 8.15ms
2014-07-11 22:11:40 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 22:11:42 get_access_token.py[line:34] INFO access_token:Kn6-tqwq3QXEayzpY4VGSW_GUudycqX24rGxqcAWhp_3EdjMc8inzVfayMp6jciAQLKHXILFJi_MgGQkFGgS8w
2014-07-11 22:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.52ms
2014-07-11 22:47:18 web.py[line:1305] ERROR Uncaught exception GET /bind/1 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='GET', uri='/bind/1', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Connection': 'close', 'Accept-Language': 'zh-cn,zh;q=0.8,en-us;q=0.5,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:30.0) Gecko/20100101 Firefox/30.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 15, in get
    self.render('login.html', time=time.ctime(), action=action)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 615, in render
    html = self.render_string(template_name, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 722, in render_string
    return t.generate(**namespace)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/template.py", line 278, in generate
    return execute()
  File "login_html.generated.py", line 5, in _tt_execute
    _tt_tmp = static_url("logo.gif")  # login.html:12
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1096, in static_url
    self.require_setting("static_path", "static_url")
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1135, in require_setting
    "application to use %s" % (name, feature))
Exception: You must define the 'static_path' setting in your application to use static_url
2014-07-11 22:47:18 web.py[line:1728] ERROR 500 GET /bind/1 (127.0.0.1) 3.94ms
2014-07-11 22:53:20 web.py[line:1728] INFO 200 GET /bind/1 (127.0.0.1) 6.80ms
2014-07-11 22:53:21 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.87ms
2014-07-11 22:55:26 web.py[line:1728] INFO 200 GET /bind/1 (127.0.0.1) 6.29ms
2014-07-11 22:55:27 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 7.33ms
2014-07-11 22:55:27 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.97ms
2014-07-11 23:11:42 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-11 23:11:44 get_access_token.py[line:34] INFO access_token:GmUdifJISZpQxGygmHHv-9HjxRWnmO_Pf1FKIjqmKkU0xOiglCsHvPw9A_0qfHqoT7PJGd6oS-TZyD71wuMfvg
2014-07-11 23:26:25 web.py[line:1728] INFO 200 GET /bind/1 (127.0.0.1) 4.67ms
2014-07-11 23:26:26 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 5.49ms
2014-07-11 23:26:26 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.79ms
2014-07-11 23:26:26 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.84ms
2014-07-11 23:26:27 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.73ms
2014-07-11 23:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.58ms
2014-07-11 23:27:06 web.py[line:1728] INFO 200 GET /bind/1 (127.0.0.1) 4.07ms
2014-07-11 23:27:07 web.py[line:1728] INFO 200 GET /bind/xiezhua_files/ga.php (127.0.0.1) 0.83ms
2014-07-11 23:27:08 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 7.40ms
2014-07-11 23:27:43 web.py[line:1728] INFO 200 GET /bind/1 (127.0.0.1) 13.20ms
2014-07-11 23:27:44 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 5.91ms
2014-07-12 00:00:25 web.py[line:1728] INFO 200 GET /bind/1 (127.0.0.1) 0.84ms
2014-07-12 00:00:25 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 0.78ms
2014-07-12 00:00:25 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.85ms
2014-07-12 00:01:39 web.py[line:1728] INFO 200 GET /bind/affasd (127.0.0.1) 0.60ms
2014-07-12 00:01:39 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.67ms
2014-07-12 00:01:39 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 1.08ms
2014-07-12 00:03:42 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 00:03:43 web.py[line:1728] INFO 200 POST /bind/1 (127.0.0.1) 1269.08ms
2014-07-12 00:11:44 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 00:11:45 get_access_token.py[line:34] INFO access_token:tz-elviMkWfpT6n8W4p9cGpTfCcsrXeoZAIoqWh4nuv0hn9ANBUBGcfnM49So_IfZBddIDxn2gORBgRoAQEu6Q
2014-07-12 00:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.47ms
2014-07-12 01:11:45 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 01:11:47 get_access_token.py[line:34] INFO access_token:a4yny7jGh1P8j0kzDYU2eox2Ah9xuTZ7KILfY_1CypqM7qsoaf69wodPJJQfJ32-TIogLY2yK0rmZ4usygHWhQ
2014-07-12 01:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.45ms
2014-07-12 02:11:47 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 02:11:49 get_access_token.py[line:34] INFO access_token:nPK6hRz4jjEGKZ-e_oKpiqy_ItueTULHtt9iAWjvImsKw8QPn2J_iE6h7KkDawYnl7dUWsJuIHHg4BDQTm6LzQ
2014-07-12 02:18:55 web.py[line:1728] INFO 200 POST /wechat?signature=638cfdbd3877bb304828053ceca773217318c55d&timestamp=1405102735&nonce=1402726937 (127.0.0.1) 2.95ms
2014-07-12 02:19:59 web.py[line:1728] INFO 200 GET /bind/oDVlVuPR4YXzgVDRW_ZI0HDrj8Xc (127.0.0.1) 0.64ms
2014-07-12 02:20:00 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.72ms
2014-07-12 02:20:00 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 0.71ms
2014-07-12 02:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.41ms
2014-07-12 03:11:49 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 03:11:50 get_access_token.py[line:34] INFO access_token:hHl7vIBCgsv8ho3zTKAB3Kxu7VWrcqLRugg5DrM9Bq7dFChuKdU2-HuVEYo6WL3C4yqVy9NIF7ycnIcS1vZQOw
2014-07-12 03:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-12 04:11:50 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 04:11:52 get_access_token.py[line:34] INFO access_token:JsnM1iFaFNFGuAfUFG-RfF-gw4d9Ms7wtHdlbgcab6HLviB1Ubf4nJt-0WqxbgxDJE95BzwruaZ8h7Awqh5w0g
2014-07-12 04:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.37ms
2014-07-12 05:11:52 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 05:11:54 get_access_token.py[line:34] INFO access_token:G9gfwShLu3XI1sDbY0lja_MkDit8t0-lbZ1PeXcrn9QNCEPpzwSJ0AU3EM9mnDe6JnZROa6r6hhKep44LA2ILA
2014-07-12 05:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-12 06:11:54 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 06:11:56 get_access_token.py[line:34] INFO access_token:2xyzA7zvm9qTzvWttQLM-IY2Q7rxH4ogeHcg44A_-gROmO8CnyRxEr2URSui8Xc9eD7Cf_NxhkEC7b_LhdBa_g
2014-07-12 06:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.40ms
2014-07-12 07:11:56 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 07:11:58 get_access_token.py[line:34] INFO access_token:JG_zOEJlcjrt-25rmgJaWpuWDgZe4SomtNnP6dwQGaFSIuGDEjguXPxAIEgKYQfTjpAjr8kXdSUeWRfJklZ0Wg
2014-07-12 07:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.56ms
2014-07-12 08:11:58 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 08:11:59 get_access_token.py[line:34] INFO access_token:hHl7vIBCgsv8ho3zTKAB3EaaPEAS7zKfRhfdsdcA0Pa4qFj6BvIqWlK1SYpLO0aHBtwKHYv_WEygNmdaWBVN9w
2014-07-12 08:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-12 09:03:09 web.py[line:1728] WARNING 404 GET /bind (127.0.0.1) 0.51ms
2014-07-12 09:03:09 web.py[line:1728] WARNING 404 GET /bind (127.0.0.1) 0.37ms
2014-07-12 09:03:14 web.py[line:1728] INFO 200 GET /bind/222 (127.0.0.1) 0.54ms
2014-07-12 09:03:14 web.py[line:1728] INFO 200 GET /bind/222 (127.0.0.1) 0.73ms
2014-07-12 09:03:16 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.68ms
2014-07-12 09:03:16 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 0.57ms
2014-07-12 09:11:59 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 09:12:01 get_access_token.py[line:34] INFO access_token:JG_zOEJlcjrt-25rmgJaWgQJ_qcm8sDAnNfAKC9RcxbLve2WFxXmuR6i1E48Zd7dpLRlidVlRJK99UH49rkK-g
2014-07-12 09:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-12 10:12:01 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 10:12:02 get_access_token.py[line:34] INFO access_token:vYtWz0_T0jM-iEZESIimM3VHc4BNaXC1jqqrxUvagx8zEfv-NFqoIwARuWBEeYDDfSEsBe5t7gWR1793NXcXXg
2014-07-12 10:22:22 web.py[line:1728] INFO 200 GET /bind/1111asdfhaj;ksldfjha;sdfjhsadl;fg (127.0.0.1) 0.54ms
2014-07-12 10:22:23 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.81ms
2014-07-12 10:22:23 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.53ms
2014-07-12 10:23:04 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.06ms
2014-07-12 10:23:04 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.81ms
2014-07-12 10:23:05 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 0.93ms
2014-07-12 10:23:42 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 10:23:44 login.py[line:38] ERROR No JSON object could be decoded
2014-07-12 10:23:44 web.py[line:1305] ERROR Uncaught exception POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Origin': 'http://bandwagon.shijunzi.info', 'Content-Length': '44', 'Accept-Language': 'zh-cn', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 6_1_3 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Mobile/10B329 MicroMessenger/5.3', 'Connection': 'close', 'Referer': 'http://bandwagon.shijunzi.info/elfin/bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM', 'Content-Type': 'application/x-www-form-urlencoded'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 40, in post
    self.render('login.html', info=hanzi.ERR_PSW, action=action)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 615, in render
    html = self.render_string(template_name, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 722, in render_string
    return t.generate(**namespace)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/template.py", line 278, in generate
    return execute()
  File "login_html.generated.py", line 11, in _tt_execute
    _tt_tmp = time  # login.html:28
NameError: global name 'time' is not defined
2014-07-12 10:23:44 web.py[line:1728] ERROR 500 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1186.16ms
2014-07-12 10:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.43ms
2014-07-12 10:36:07 web.py[line:1728] INFO 200 POST /wechat?signature=94c9609764ab1225ae8410afebcdebf4129f6499&timestamp=1405132567&nonce=1172501244 (127.0.0.1) 6.56ms
2014-07-12 10:36:09 web.py[line:1728] INFO 200 POST /wechat?signature=31d19b572e1eb14c01960c5bca13e570fad474f8&timestamp=1405132569&nonce=2069760257 (127.0.0.1) 1.29ms
2014-07-12 10:36:17 web.py[line:1728] INFO 200 POST /wechat?signature=6edcb7dc3d6994fde5284025188e6691e5bcc6a5&timestamp=1405132575&nonce=1773800000 (127.0.0.1) 1.26ms
2014-07-12 10:36:21 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 2.46ms
2014-07-12 10:36:47 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 10:36:49 web.py[line:1728] INFO 200 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1023.54ms
2014-07-12 10:37:04 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=6a5cd995b888ac8e87e7e3f8a6207f10f56a9e22&timestamp=1405132623&nonce=661240583 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=6a5cd995b888ac8e87e7e3f8a6207f10f56a9e22&timestamp=1405132623&nonce=661240583', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
NameError: global name 'send' is not defined
2014-07-12 10:37:04 web.py[line:1728] ERROR 500 POST /wechat?signature=6a5cd995b888ac8e87e7e3f8a6207f10f56a9e22&timestamp=1405132623&nonce=661240583 (127.0.0.1) 3.01ms
2014-07-12 10:41:22 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=50cac87b80728f357ce26ab0dbeddf66c0845e44&timestamp=1405132882&nonce=1805014059 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=50cac87b80728f357ce26ab0dbeddf66c0845e44&timestamp=1405132882&nonce=1805014059', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 76, in send
    logging(user['id'] + ' : ' + talk)
NameError: global name 'logging' is not defined
2014-07-12 10:41:22 web.py[line:1728] ERROR 500 POST /wechat?signature=50cac87b80728f357ce26ab0dbeddf66c0845e44&timestamp=1405132882&nonce=1805014059 (127.0.0.1) 12.15ms
2014-07-12 10:51:44 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=3cd56d4c0406f845a9719d93670da770049690eb&timestamp=1405133504&nonce=1241914981 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=3cd56d4c0406f845a9719d93670da770049690eb&timestamp=1405133504&nonce=1241914981', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '282', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 76, in send
    logging(user['id'] + ' : ' + talk)
NameError: global name 'logging' is not defined
2014-07-12 10:51:44 web.py[line:1728] ERROR 500 POST /wechat?signature=3cd56d4c0406f845a9719d93670da770049690eb&timestamp=1405133504&nonce=1241914981 (127.0.0.1) 7.57ms
2014-07-12 10:53:31 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=773807b37620040e08828b2b59aae6556ad615c7&timestamp=1405133611&nonce=1558825356 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=773807b37620040e08828b2b59aae6556ad615c7&timestamp=1405133611&nonce=1558825356', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 76, in send
    logging(user['id'] + ' : ' + talk)
TypeError: 'module' object is not callable
2014-07-12 10:53:31 web.py[line:1728] ERROR 500 POST /wechat?signature=773807b37620040e08828b2b59aae6556ad615c7&timestamp=1405133611&nonce=1558825356 (127.0.0.1) 5.97ms
2014-07-12 10:56:37 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=c54131b52305de6f76dac943ec3a1970dda62399&timestamp=1405133796&nonce=2136288527 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=c54131b52305de6f76dac943ec3a1970dda62399&timestamp=1405133796&nonce=2136288527', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '282', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 76, in send
    log(user['id'] + ' : ' + talk)
TypeError: 'NoneType' object is not callable
2014-07-12 10:56:37 web.py[line:1728] ERROR 500 POST /wechat?signature=c54131b52305de6f76dac943ec3a1970dda62399&timestamp=1405133796&nonce=2136288527 (127.0.0.1) 6.12ms
2014-07-12 11:02:46 <stdin>[line:1] INFO hello
2014-07-12 11:04:39 send_talk.py[line:77] INFO 47637 : hello
2014-07-12 11:04:39 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=34052ec3e35fdc4c5dbc89938150bfe0f52d44fb&timestamp=1405134279&nonce=259577102 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=34052ec3e35fdc4c5dbc89938150bfe0f52d44fb&timestamp=1405134279&nonce=259577102', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 58, in send
    except SensitiveError:
NameError: global name 'SensitiveError' is not defined
2014-07-12 11:04:39 web.py[line:1728] ERROR 500 POST /wechat?signature=34052ec3e35fdc4c5dbc89938150bfe0f52d44fb&timestamp=1405134279&nonce=259577102 (127.0.0.1) 7.00ms
2014-07-12 11:12:02 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 11:12:04 get_access_token.py[line:34] INFO access_token:aYKMyfY_Uzn2U0G94DUmUKuW5QeOW0WKxPbc0ZBvuCOh-B47hcLBJoRyc_CfNjKVB8c6HFuQCzdVFWvwlxN5NQ
2014-07-12 11:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.43ms
2014-07-12 11:59:43 send_talk.py[line:73] INFO 47637 : hello
2014-07-12 11:59:43 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=eb43f1b4f3cbcc4f287f7644f70bbd92f630caaf&timestamp=1405137583&nonce=1193338790 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=eb43f1b4f3cbcc4f287f7644f70bbd92f630caaf&timestamp=1405137583&nonce=1193338790', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 58, in send
    except OverLengthError:
NameError: global name 'OverLengthError' is not defined
2014-07-12 11:59:43 web.py[line:1728] ERROR 500 POST /wechat?signature=eb43f1b4f3cbcc4f287f7644f70bbd92f630caaf&timestamp=1405137583&nonce=1193338790 (127.0.0.1) 18.72ms
2014-07-12 12:12:04 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 12:12:05 get_access_token.py[line:34] INFO access_token:mLUhCTEER1BvzJdaGNU9CwgxtVY7xoTePZCCRFM35q5nIgKm_WR4cIiT5XqeZuw1ogGpmElhaih6EzEPTNceJQ
2014-07-12 12:15:40 send_talk.py[line:77] INFO 47637 : hello
2014-07-12 12:15:40 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=82009f8137b3cab78792c178954ebbc5d41d4e6c&timestamp=1405138539&nonce=836034512 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=82009f8137b3cab78792c178954ebbc5d41d4e6c&timestamp=1405138539&nonce=836034512', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 67, in send
    send_ok_ret = transmit(user, talk)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 38, in transmit
    data['source'] = user['tail'].decode('utf-8').encode('GB18030')
  File "/home/nightwish/elfin/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-4: ordinal not in range(128)
2014-07-12 12:15:40 web.py[line:1728] ERROR 500 POST /wechat?signature=82009f8137b3cab78792c178954ebbc5d41d4e6c&timestamp=1405138539&nonce=836034512 (127.0.0.1) 8.64ms
2014-07-12 12:17:12 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 12:17:13 send_talk.py[line:77] INFO 47637 : hello
2014-07-12 12:17:13 web.py[line:1728] INFO 200 POST /wechat?signature=48cd5c2cdac3489b5ed3e86eb37139c85cad052c&timestamp=1405138630&nonce=1715644649 (127.0.0.1) 1092.57ms
2014-07-12 12:17:17 web.py[line:1302] WARNING 400 GET /wechat (127.0.0.1): Missing argument signature
2014-07-12 12:17:17 web.py[line:1728] WARNING 400 GET /wechat (127.0.0.1) 1.27ms
2014-07-12 12:18:05 send_talk.py[line:77] INFO 47637 : 共和军
2014-07-12 12:18:05 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=903355526d06d796701f3808d4b31a66f2eeee74&timestamp=1405138685&nonce=2119668459 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=903355526d06d796701f3808d4b31a66f2eeee74&timestamp=1405138685&nonce=2119668459', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '282', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 67, in send
    send_ok_ret = transmit(user, talk)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 37, in transmit
    data['status'] = talk.decode('utf-8').encode('GB18030')
  File "/home/nightwish/elfin/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128)
2014-07-12 12:18:05 web.py[line:1728] ERROR 500 POST /wechat?signature=903355526d06d796701f3808d4b31a66f2eeee74&timestamp=1405138685&nonce=2119668459 (127.0.0.1) 5.23ms
2014-07-12 12:18:52 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 12:18:53 send_talk.py[line:77] INFO 47637 : 复活节
2014-07-12 12:18:53 web.py[line:1728] INFO 200 POST /wechat?signature=09778bb17d1308e6dff9fd171ab40446e10b251d&timestamp=1405138732&nonce=1989094938 (127.0.0.1) 1130.24ms
2014-07-12 12:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.53ms
2014-07-12 12:57:28 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 12:57:30 send_talk.py[line:77] INFO 47637 : 很少见到
2014-07-12 12:57:30 web.py[line:1728] INFO 200 POST /wechat?signature=e9bd3754f59f31af37ed4021f20396c828f60025&timestamp=1405141048&nonce=1836985067 (127.0.0.1) 2140.89ms
2014-07-12 13:00:48 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:00:49 send_talk.py[line:77] INFO 47637 : 搞活经济
2014-07-12 13:00:49 web.py[line:1728] INFO 200 POST /wechat?signature=7037d52e4fd7de152c30decb9e016ef8547b6c46&timestamp=1405141246&nonce=972090832 (127.0.0.1) 1008.84ms
2014-07-12 13:05:09 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:05:10 send_talk.py[line:79] INFO 47637 : 骶棘肌蒂
2014-07-12 13:05:10 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=74b9290a8c2ceae50006a7be30fa0ac606aff99d&timestamp=1405141507&nonce=1716370225 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=74b9290a8c2ceae50006a7be30fa0ac606aff99d&timestamp=1405141507&nonce=1716370225', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '285', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 68, in send
    send_ok_ret = transmit(user, talk)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 51, in transmit
    send_ok_ret(user)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 30, in send_ok_ret
    logging.info('ret_state:'+ret_state)
TypeError: cannot concatenate 'str' and 'int' objects
2014-07-12 13:05:10 web.py[line:1728] ERROR 500 POST /wechat?signature=74b9290a8c2ceae50006a7be30fa0ac606aff99d&timestamp=1405141507&nonce=1716370225 (127.0.0.1) 1723.09ms
2014-07-12 13:06:02 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:06:04 send_talk.py[line:30] INFO ret_state:1
2014-07-12 13:06:04 send_talk.py[line:79] INFO 47637 : 顶焦度计
2014-07-12 13:06:04 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=7272c1d755f54692e0332199e406e5b4866a2ecc&timestamp=1405141562&nonce=573257264 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=7272c1d755f54692e0332199e406e5b4866a2ecc&timestamp=1405141562&nonce=573257264', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '285', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 76, in send
    logging.info('send_ok_ret:'+send_ok_ret)
TypeError: cannot concatenate 'str' and 'NoneType' objects
2014-07-12 13:06:04 web.py[line:1728] ERROR 500 POST /wechat?signature=7272c1d755f54692e0332199e406e5b4866a2ecc&timestamp=1405141562&nonce=573257264 (127.0.0.1) 2235.37ms
2014-07-12 13:09:49 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:09:50 send_talk.py[line:30] INFO ret_state:1
2014-07-12 13:09:50 send_talk.py[line:79] INFO 47637 : 顶焦度计
2014-07-12 13:09:50 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=3761f6f8f675e95dd09688839d1b35b9cc10cd37&timestamp=1405141788&nonce=2030470542 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=3761f6f8f675e95dd09688839d1b35b9cc10cd37&timestamp=1405141788&nonce=2030470542', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '285', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 76, in send
    logging.info('send_ok_ret:'+send_ok_ret)
TypeError: cannot concatenate 'str' and 'NoneType' objects
2014-07-12 13:09:50 web.py[line:1728] ERROR 500 POST /wechat?signature=3761f6f8f675e95dd09688839d1b35b9cc10cd37&timestamp=1405141788&nonce=2030470542 (127.0.0.1) 916.75ms
2014-07-12 13:11:06 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:11:07 send_talk.py[line:30] INFO ret_state:1
2014-07-12 13:11:07 send_talk.py[line:76] INFO send_ok_ret:ok
2014-07-12 13:11:07 send_talk.py[line:79] INFO 47637 : 大男大女
2014-07-12 13:11:07 web.py[line:1728] INFO 200 POST /wechat?signature=a2101237100eca484f9d0da81048a353f481c648&timestamp=1405141866&nonce=251398690 (127.0.0.1) 908.69ms
2014-07-12 13:12:06 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 13:12:07 get_access_token.py[line:34] INFO access_token:KGUNJxdsMQ-DYGo8E4YJBJHGy5y9KXLnAgko_Troa9iBh8vDdV5xFG7VPkvjbyqiUwjIXF4tbq3KCaBEnhTLTQ
2014-07-12 13:13:25 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:13:26 send_talk.py[line:30] INFO ret_state:0
2014-07-12 13:13:26 send_talk.py[line:76] INFO send_ok_ret:
2014-07-12 13:13:26 send_talk.py[line:79] INFO 47637 : 大哭大闹
2014-07-12 13:13:26 web.py[line:1728] INFO 200 POST /wechat?signature=cf40fda77a2ec77288c8cb76e92df575c87877a5&timestamp=1405142005&nonce=240115909 (127.0.0.1) 1037.53ms
2014-07-12 13:13:34 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:13:35 send_talk.py[line:30] INFO ret_state:0
2014-07-12 13:13:35 send_talk.py[line:76] INFO send_ok_ret:
2014-07-12 13:13:35 send_talk.py[line:79] INFO 47637 : v免费看到
2014-07-12 13:13:35 web.py[line:1728] INFO 200 POST /wechat?signature=ef0cdc5f31345d12dc9ece4fd6137abb85ca9f2f&timestamp=1405142014&nonce=1113584690 (127.0.0.1) 1218.19ms
2014-07-12 13:15:22 send_talk.py[line:79] INFO 47637 : sxjzz7wIyel
2014-07-12 13:15:22 web.py[line:1728] INFO 200 POST /wechat?signature=5e13bc98a0cf40f833b0a0001eb8c016bf5ecf7f&timestamp=1405142121&nonce=32521135 (127.0.0.1) 2.77ms
2014-07-12 13:15:31 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:15:34 send_talk.py[line:30] INFO ret_state:0
2014-07-12 13:15:34 send_talk.py[line:76] INFO send_ok_ret:
2014-07-12 13:15:34 send_talk.py[line:79] INFO 47637 : 1
2014-07-12 13:15:34 web.py[line:1728] INFO 200 POST /wechat?signature=ceef2327de209223fed692523912ce6017878c44&timestamp=1405142130&nonce=1821805502 (127.0.0.1) 2701.98ms
2014-07-12 13:15:34 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:15:35 send_talk.py[line:30] INFO ret_state:0
2014-07-12 13:15:35 send_talk.py[line:76] INFO send_ok_ret:
2014-07-12 13:15:35 send_talk.py[line:79] INFO 47637 : 1
2014-07-12 13:15:35 web.py[line:1728] INFO 200 POST /wechat?signature=79b54c754d5c1aaa8b7635a3f76fb859ac6360c3&timestamp=1405142133&nonce=1377766003 (127.0.0.1) 1056.26ms
2014-07-12 13:22:25 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:22:28 send_talk.py[line:78] INFO 47637 : 1
2014-07-12 13:22:28 web.py[line:1728] INFO 200 POST /wechat?signature=2a8b607382acc9edf24a672f460c939c904e6570&timestamp=1405142545&nonce=1265405137 (127.0.0.1) 2139.70ms
2014-07-12 13:22:33 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:22:34 send_talk.py[line:78] INFO 47637 : 1
2014-07-12 13:22:34 web.py[line:1728] INFO 200 POST /wechat?signature=2099bedfad40f94c2bbe37df1abc5044b822ef3b&timestamp=1405142552&nonce=1090987692 (127.0.0.1) 904.34ms
2014-07-12 13:23:17 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:23:18 send_talk.py[line:78] INFO 47637 : 1
2014-07-12 13:23:18 web.py[line:1728] INFO 200 POST /wechat?signature=55cd57ce250982c1f9435a9dff9c96ccfaaf1f1f&timestamp=1405142597&nonce=1437257183 (127.0.0.1) 985.69ms
2014-07-12 13:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.40ms
2014-07-12 13:27:51 send_talk.py[line:81] INFO 47637 : 1
2014-07-12 13:27:51 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=b3c194503e81b709306c5df215569f3299d2389e&timestamp=1405142868&nonce=1891642539 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=b3c194503e81b709306c5df215569f3299d2389e&timestamp=1405142868&nonce=1891642539', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '274', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 59, in send
    logging.info('old hash:'+user['hash'])
TypeError: cannot concatenate 'str' and 'int' objects
2014-07-12 13:27:51 web.py[line:1728] ERROR 500 POST /wechat?signature=b3c194503e81b709306c5df215569f3299d2389e&timestamp=1405142868&nonce=1891642539 (127.0.0.1) 9.77ms
2014-07-12 13:30:11 send_talk.py[line:59] INFO old hash:1977051568
2014-07-12 13:30:11 send_talk.py[line:22] INFO new hash:1977051568
2014-07-12 13:30:11 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:30:12 send_talk.py[line:81] INFO 47637 : 1
2014-07-12 13:30:12 web.py[line:1728] INFO 200 POST /wechat?signature=f83c392fa38e297c7819c7688748ed67ff2b2867&timestamp=1405143010&nonce=1380752465 (127.0.0.1) 1062.17ms
2014-07-12 13:30:35 send_talk.py[line:59] INFO old hash:1977051568
2014-07-12 13:30:35 send_talk.py[line:22] INFO new hash:1977051568
2014-07-12 13:30:35 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:30:36 send_talk.py[line:81] INFO 47637 : 1
2014-07-12 13:30:36 web.py[line:1728] INFO 200 POST /wechat?signature=290d6a65aae5ceb5e334a006aa27a3a47507f899&timestamp=1405143034&nonce=400137183 (127.0.0.1) 1017.81ms
2014-07-12 13:37:28 send_talk.py[line:59] INFO old hash:1977051568
2014-07-12 13:37:28 send_talk.py[line:22] INFO new hash:1977051568
2014-07-12 13:37:28 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:37:29 send_talk.py[line:81] INFO 47637 : 1
2014-07-12 13:37:29 web.py[line:1728] INFO 200 POST /wechat?signature=4bf277546b7dcb22c7ba3a743ee3d572bee7b17c&timestamp=1405143448&nonce=1757745612 (127.0.0.1) 966.60ms
2014-07-12 13:42:36 send_talk.py[line:59] INFO old hash:1977051568
2014-07-12 13:42:36 send_talk.py[line:81] INFO 47637 : 1
2014-07-12 13:42:36 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=d949b2e861c1c3c08a5d1a7e2a591bd4f059b725&timestamp=1405143756&nonce=1358856187 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=d949b2e861c1c3c08a5d1a7e2a591bd4f059b725&timestamp=1405143756&nonce=1358856187', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '274', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 42, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 60, in send
    diff_from_old(talk, user['hash'])
  File "/home/nightwish/elfin/scripts/send_talk.py", line 22, in diff_from_old
    logging.info('new hash:'+str(new_hash)+ ' '+type(new_hash)+' '+type(old_hash))
TypeError: cannot concatenate 'str' and 'type' objects
2014-07-12 13:42:36 web.py[line:1728] ERROR 500 POST /wechat?signature=d949b2e861c1c3c08a5d1a7e2a591bd4f059b725&timestamp=1405143756&nonce=1358856187 (127.0.0.1) 8.85ms
2014-07-12 13:43:50 send_talk.py[line:59] INFO old hash:1977051568
2014-07-12 13:43:50 send_talk.py[line:22] INFO new hash:1977051568 <type 'int'> <type 'int'>
2014-07-12 13:43:50 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:43:51 send_talk.py[line:81] INFO 47637 : 1
2014-07-12 13:43:51 web.py[line:1728] INFO 200 POST /wechat?signature=8741acd8e66d2097e595b8122be53e8acd752162&timestamp=1405143830&nonce=568826125 (127.0.0.1) 1003.51ms
2014-07-12 13:51:03 send_talk.py[line:22] INFO 1977051568
2014-07-12 13:51:03 send_talk.py[line:23] INFO <type 'int'>
2014-07-12 13:51:03 send_talk.py[line:24] INFO 1977051568
2014-07-12 13:51:03 send_talk.py[line:25] INFO <type 'int'>
2014-07-12 13:51:03 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:51:05 send_talk.py[line:83] INFO 47637 : 1
2014-07-12 13:51:05 web.py[line:1728] INFO 200 POST /wechat?signature=e752a8dde08a19687cb24d034f5a4cf242525e9c&timestamp=1405144262&nonce=178866659 (127.0.0.1) 1909.64ms
2014-07-12 13:54:26 send_talk.py[line:65] INFO 请勿发送密码
2014-07-12 13:54:26 send_talk.py[line:83] INFO 47637 : sxjzz7wIyel
2014-07-12 13:54:26 web.py[line:1728] INFO 200 POST /wechat?signature=0eea01d7566bf0d6da05b73e515856850b95091e&timestamp=1405144466&nonce=819516542 (127.0.0.1) 2.95ms
2014-07-12 13:54:52 send_talk.py[line:22] INFO 1977051568
2014-07-12 13:54:52 send_talk.py[line:23] INFO <type 'int'>
2014-07-12 13:54:52 send_talk.py[line:24] INFO 1977051568
2014-07-12 13:54:52 send_talk.py[line:25] INFO <type 'int'>
2014-07-12 13:54:52 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 13:54:53 send_talk.py[line:83] INFO 47637 : 1
2014-07-12 13:54:53 web.py[line:1728] INFO 200 POST /wechat?signature=64fbe5b2a04fc7ea2a85d268a0b60edaac201442&timestamp=1405144492&nonce=776221042 (127.0.0.1) 891.05ms
2014-07-12 13:59:04 send_talk.py[line:22] INFO 1977051568
2014-07-12 13:59:04 send_talk.py[line:23] INFO <type 'int'>
2014-07-12 13:59:04 send_talk.py[line:24] INFO 1977051568
2014-07-12 13:59:04 send_talk.py[line:25] INFO <type 'int'>
2014-07-12 13:59:04 send_talk.py[line:66] INFO 请勿重复发送
2014-07-12 13:59:04 send_talk.py[line:84] INFO 47637 : 1
2014-07-12 13:59:04 web.py[line:1728] INFO 200 POST /wechat?signature=eb8896cf61410c19034cc5c786bebee706ee8998&timestamp=1405144743&nonce=320817019 (127.0.0.1) 10.42ms
2014-07-12 14:07:19 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 14:07:20 send_talk.py[line:79] INFO 47637 : Fight
2014-07-12 14:07:20 web.py[line:1728] INFO 200 POST /wechat?signature=cb46da75dd954f62c7dc7f5eccb6566de541d18f&timestamp=1405145238&nonce=1496399076 (127.0.0.1) 1000.23ms
2014-07-12 14:08:32 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-12 14:08:33 send_talk.py[line:79] INFO 47637 : Hjh
2014-07-12 14:08:33 web.py[line:1728] INFO 200 POST /wechat?signature=b11afae2f71837d889e462e9c3d51b81d6186c8e&timestamp=1405145311&nonce=15149668 (127.0.0.1) 882.37ms
2014-07-12 14:12:07 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 14:12:08 get_access_token.py[line:34] INFO access_token:aYKMyfY_Uzn2U0G94DUmUPDpatSXAje8VK6AG0nVVFyrXfr11wKAg6RFnCihSfs0MAlW-QUAr0yltCi8zPNUww
2014-07-12 14:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-12 14:29:34 web.py[line:1728] INFO 200 GET /bind/2345 (127.0.0.1) 4.49ms
2014-07-12 14:29:35 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 5.70ms
2014-07-12 14:29:35 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 0.62ms
2014-07-12 14:30:09 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.79ms
2014-07-12 14:30:15 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.75ms
2014-07-12 14:30:23 web.py[line:1728] INFO 200 GET /static/logo.gif (127.0.0.1) 0.75ms
2014-07-12 15:12:08 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 15:12:10 get_access_token.py[line:34] INFO access_token:QiPWVoyDZHXk9NughCxpDdxF1nnKFm-Vi-C9XwpIntAfDp_zl9fotKdjgxYa6Jc2sA-l3OlxBWkRMWS2RpSamw
2014-07-12 15:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.55ms
2014-07-12 15:52:25 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 7.17ms
2014-07-12 15:52:25 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 5.84ms
2014-07-12 15:52:26 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.82ms
2014-07-12 15:52:38 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.83ms
2014-07-12 15:52:38 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 1.00ms
2014-07-12 15:52:39 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.78ms
2014-07-12 15:53:42 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.28ms
2014-07-12 15:53:43 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 0.93ms
2014-07-12 15:53:43 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.78ms
2014-07-12 15:55:16 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 0.80ms
2014-07-12 15:55:16 web.py[line:1728] INFO 200 GET /static/login.css (127.0.0.1) 0.76ms
2014-07-12 15:55:16 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.52ms
2014-07-12 16:12:10 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 16:12:12 get_access_token.py[line:34] INFO access_token:7YtYD01ErTJEG4p6Fah7mcte3ML__LCPlRWEiNLdBz2n1EcA0CZvvP9CIJTylK_k2cafnHy4mIt4c3SEEjEOVw
2014-07-12 16:16:14 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 0.75ms
2014-07-12 16:16:14 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 0.80ms
2014-07-12 16:16:29 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.59ms
2014-07-12 16:16:29 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 0.59ms
2014-07-12 16:16:30 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.71ms
2014-07-12 16:16:35 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.81ms
2014-07-12 16:16:35 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 0.74ms
2014-07-12 16:16:36 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.71ms
2014-07-12 16:16:47 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.19ms
2014-07-12 16:16:48 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 1.01ms
2014-07-12 16:16:49 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.01ms
2014-07-12 16:16:56 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 1.11ms
2014-07-12 16:16:57 web.py[line:1728] INFO 304 GET /static/login.css (127.0.0.1) 0.72ms
2014-07-12 16:17:32 web.py[line:1305] ERROR Uncaught exception GET /tail/1111 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='GET', uri='/tail/1111', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Connection': 'close', 'Accept-Language': 'en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4', 'Accept-Encoding': 'gzip,deflate,sdch', 'If-None-Match': '"60cf269be0fd8304c4b36e5b6522b2c710041dda"', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/tail.py", line 16, in get
    self.render('tail.html', action=action)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 615, in render
    html = self.render_string(template_name, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 722, in render_string
    return t.generate(**namespace)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/template.py", line 278, in generate
    return execute()
  File "tail_html.generated.py", line 13, in _tt_execute
    _tt_tmp = time  # base.html:22
NameError: global name 'time' is not defined
2014-07-12 16:17:32 web.py[line:1728] ERROR 500 GET /tail/1111 (127.0.0.1) 7.22ms
2014-07-12 16:19:03 web.py[line:1728] INFO 200 GET /tail/1111 (127.0.0.1) 5.69ms
2014-07-12 16:19:03 web.py[line:1728] WARNING 404 GET /static/base.css (127.0.0.1) 0.68ms
2014-07-12 16:19:40 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 4.69ms
2014-07-12 16:19:40 web.py[line:1728] WARNING 404 GET /static/base.css (127.0.0.1) 0.94ms
2014-07-12 16:21:27 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 0.59ms
2014-07-12 16:21:27 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 3.96ms
2014-07-12 16:21:38 web.py[line:1728] INFO 200 GET /tail/1111 (127.0.0.1) 0.85ms
2014-07-12 16:21:38 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.76ms
2014-07-12 16:21:52 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.00ms
2014-07-12 16:21:53 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 0.94ms
2014-07-12 16:21:53 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.78ms
2014-07-12 16:22:17 web.py[line:1728] INFO 200 GET /tail/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.79ms
2014-07-12 16:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-12 16:40:02 web.py[line:1728] INFO 200 GET /tail/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 19.77ms
2014-07-12 16:40:02 web.py[line:1728] INFO 200 GET /static/jquery.min.js (127.0.0.1) 6.20ms
2014-07-12 16:40:03 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.94ms
2014-07-12 16:40:03 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.15ms
2014-07-12 16:40:19 web.py[line:1728] INFO 200 GET /tail/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.84ms
2014-07-12 16:40:19 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.90ms
2014-07-12 16:40:19 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.65ms
2014-07-12 16:40:20 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.06ms
2014-07-12 16:40:34 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 2.97ms
2014-07-12 16:40:48 web.py[line:1728] INFO 200 GET /tail/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.76ms
2014-07-12 16:40:49 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.71ms
2014-07-12 17:12:13 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 17:12:14 get_access_token.py[line:34] INFO access_token:2RRuM7nb1nVLxtKQcQJYohEmLPTeOV1gvRz9HQXs3E7Zr73JaADSZTG9XGKmog9iJ_vlqL9xqsaAHHCMNilo6g
2014-07-12 17:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.59ms
2014-07-12 17:48:25 web.py[line:1728] INFO 200 GET /tail/1111 (127.0.0.1) 5.12ms
2014-07-12 17:48:25 web.py[line:1728] INFO 200 GET /static/jquery.min.js (127.0.0.1) 10.31ms
2014-07-12 17:48:26 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 1.04ms
2014-07-12 17:48:26 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.69ms
2014-07-12 17:48:38 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 2.06ms
2014-07-12 17:48:39 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.82ms
2014-07-12 17:48:39 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.53ms
2014-07-12 17:49:55 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.85ms
2014-07-12 17:49:55 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.68ms
2014-07-12 17:49:56 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 0.80ms
2014-07-12 17:49:56 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.71ms
2014-07-12 18:12:14 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 18:12:16 get_access_token.py[line:34] INFO access_token:hSZOyKHwHGovNExkjllNQsmgBHOi5emAcNfo0YCRLb77iRBDJMZHzgjK9kSrzsxoigZewrmPiWYjXPdPz5NuUw
2014-07-12 18:12:19 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 0.59ms
2014-07-12 18:12:19 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.60ms
2014-07-12 18:12:19 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.64ms
2014-07-12 18:26:48 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 5.33ms
2014-07-12 18:26:48 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 6.04ms
2014-07-12 18:26:49 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 1.14ms
2014-07-12 18:26:49 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.24ms
2014-07-12 18:26:53 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.75ms
2014-07-12 18:26:53 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.90ms
2014-07-12 18:26:53 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.73ms
2014-07-12 18:26:54 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.66ms
2014-07-12 18:26:56 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.77ms
2014-07-12 18:26:57 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 1.04ms
2014-07-12 18:26:57 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 1.06ms
2014-07-12 18:26:57 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.10ms
2014-07-12 18:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.52ms
2014-07-12 18:38:04 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 5.85ms
2014-07-12 18:38:05 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 7.22ms
2014-07-12 18:38:05 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 1.18ms
2014-07-12 18:38:06 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.03ms
2014-07-12 18:38:21 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1.13ms
2014-07-12 18:38:21 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 1.00ms
2014-07-12 18:38:26 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.95ms
2014-07-12 18:38:27 web.py[line:1728] INFO 304 GET /static/base.css (127.0.0.1) 0.81ms
2014-07-12 18:38:27 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.76ms
2014-07-12 18:38:27 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.99ms
2014-07-12 18:38:45 web.py[line:1728] INFO 200 GET /bind/1111 (127.0.0.1) 0.85ms
2014-07-12 18:38:45 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 0.74ms
2014-07-12 18:38:45 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 0.69ms
2014-07-12 18:39:35 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 5.82ms
2014-07-12 18:39:36 web.py[line:1728] INFO 304 GET /static/jquery.min.js (127.0.0.1) 10.02ms
2014-07-12 18:39:36 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 1.24ms
2014-07-12 18:39:36 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 1.09ms
2014-07-12 18:47:27 web.py[line:1728] INFO 200 GET /bind/1000 (127.0.0.1) 5.42ms
2014-07-12 19:12:16 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 19:12:17 get_access_token.py[line:34] INFO access_token:3A0WwQZvyPmwyrAWejMe-Hk87HQfcK9eTZOdMuKHqzTO9bTBlO0QXluGxKYajD5L0tnR0VdcRMv_7jj8JcQZKg
2014-07-12 19:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.40ms
2014-07-12 20:12:18 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 20:12:19 get_access_token.py[line:34] INFO access_token:Kn6-tqwq3QXEayzpY4VGSZOh_7zKMyHvaY_-bfKuSIC1yuDZR_I19tyUdyesqEkDx2AY6NzcJBLJF0-MeLaKtA
2014-07-12 20:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-12 21:12:19 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 21:12:21 get_access_token.py[line:34] INFO access_token:JsnM1iFaFNFGuAfUFG-RfArDSvwSUXwc_5oqdJbgN3om64QpaXxIPma-RHt4urjIeukiG0wAL4PYZHmMdi6yGQ
2014-07-12 21:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-12 22:12:21 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 22:12:22 get_access_token.py[line:34] INFO access_token:R0PqbLSYokXKz0iKdpqyaaH5AtguJJLLQ37xuOd2PDtYXKZWQsyJqogbr_41DeouwtwD38A4m1yIq1YNtlBKfg
2014-07-12 22:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 2.49ms
2014-07-12 23:12:23 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-12 23:12:24 get_access_token.py[line:34] INFO access_token:KCzwFqsFyIwQ7HYlx5OzfPwN0iw7bjWJMA3TAqpfg3ILVFWocc73Ttn9Sn15PqDps9tVjZl0uga_NeMNUQU79A
2014-07-12 23:27:07 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.51ms
2014-07-13 00:12:24 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 00:12:26 get_access_token.py[line:34] INFO access_token:8mdBiLF_rAT5pCHVbAXE6f5zxIWMZbqVOkL5ch85iexRapV_K1ChUX6zVE-Qaj5vs1RUCC0Q8e6mrJ568oEv0A
2014-07-13 00:27:06 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.58ms
2014-07-13 01:12:26 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 01:12:27 get_access_token.py[line:34] INFO access_token:R0PqbLSYokXKz0iKdpqyae9o7PLdUg6omJiR2deCQOxel4elOa0FHGah2u-dC6qQZEDIm5r-Uqo5M6CdR5oLaQ
2014-07-13 01:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.50ms
2014-07-13 02:12:27 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 02:12:29 get_access_token.py[line:34] INFO access_token:ssrRyWVmbJn5cLRh5AOw8y-d6E_6xhAe0wh9mkQknDeJ6m1Q3dIXmAneTb-mR6p5flBnhs4Bn9GFIt8vB4GAQg
2014-07-13 02:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.52ms
2014-07-13 03:12:29 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 03:12:32 get_access_token.py[line:34] INFO access_token:rEc4dmdZJQqLQqIzJCbF_lPkYGVYRVyuVlU_5LZxMNNXUba6eLEi_soDEyH-G1blTA8l0tqkHijhgvimucqsIw
2014-07-13 03:27:06 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.37ms
2014-07-13 04:12:32 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 04:12:33 get_access_token.py[line:34] INFO access_token:R0PqbLSYokXKz0iKdpqyaeXf0ewwtuLRrap6nQfF5kM4zcRiaFyI3fnScIRnr3ljYos5pVSHvtChsfrSJdSWrQ
2014-07-13 04:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.40ms
2014-07-13 05:12:33 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 05:12:35 get_access_token.py[line:34] INFO access_token:wy33zJ0l0BVjdYZdEL_2zc3T1C7E3rPq2ttHTQ83GHUl34J0_1-N-5wIPcLLCnGRgaNBEyvFm25h6DZXMVETGA
2014-07-13 05:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-13 06:12:35 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 06:12:36 get_access_token.py[line:34] INFO access_token:5njussikalv9kPpuEMHaRCalOhBEBbNqKaBKlyLL_0FvSb5JliB_Wh9OKfudMURBJaotqXSATm_WRJyYxypOGg
2014-07-13 06:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.46ms
2014-07-13 07:12:36 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 07:12:38 get_access_token.py[line:34] INFO access_token:bf7pFQqxfKv1fYyv_BTK5dm7d2nWdNqjQjIYlVsjwILIMymXPCy3xY3ALGrN7b7H0jb-xv54N6w4Lzgg7wQIsg
2014-07-13 07:27:04 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-13 08:12:38 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 08:12:39 get_access_token.py[line:34] INFO access_token:hSZOyKHwHGovNExkjllNQvCK-C4tjZdT0b3f3kUysyVDYVtyh8d13hY1bBjGJw_5HCU3zfnRSAMisEoTj0V2uA
2014-07-13 08:27:05 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.37ms
2014-07-13 09:12:39 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 09:12:41 get_access_token.py[line:34] INFO access_token:GmUdifJISZpQxGygmHHv-6LTe0o1r3cF9rehtZoDZ08cCF5D0F98Niy5LWeILG5vfFjn2AT-2ByFqZeq_chMmQ
2014-07-13 09:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-13 10:12:41 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 10:12:43 get_access_token.py[line:34] INFO access_token:EhokNy96sCG38d0y2ma7AAnbwg_GD9KaOseiTlIP4jQ3M43NUNbalDByF0opTaa400iKvqw7QZ9-iHZrD2VMTg
2014-07-13 10:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-13 10:34:09 web.py[line:1728] INFO 200 POST /wechat?signature=99bd285ea0bb8166fea8ce7242b3711690ff8238&timestamp=1405218849&nonce=423585118 (127.0.0.1) 2.74ms
2014-07-13 11:12:43 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 11:12:44 get_access_token.py[line:34] INFO access_token:L9ooAKfAO3QJ7o8q8ezLJcAZabXUJb83NJ1Et4ssCOWSM4bk40qfMOQPiiQQ8vqoqL0lk7f5idoR7SqanRCE6w
2014-07-13 11:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.41ms
2014-07-13 12:12:44 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 12:12:46 get_access_token.py[line:34] INFO access_token:R_XPsjbljvmE_Lm6iyeB0ps4fsoVLPzaYaN7X7wPk2Kuu5m6i4dkUnIUwxmhC5LhXFwVtNQq7KL-yNaI5_SP_A
2014-07-13 12:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.57ms
2014-07-13 13:12:46 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 13:12:48 get_access_token.py[line:34] INFO access_token:KY7--O7aQp6CSrLuaUblmeYCz0O5a4l4-JVwtVq_xSHs_PAPWJFCYnB7oK36je8rteXV0U9f7WmNhh3k5XfF_Q
2014-07-13 13:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-13 14:12:48 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 14:12:50 get_access_token.py[line:34] INFO access_token:x_ZIZvbitNHTRLQG2XlWGhs3-wqM5uzMSwj6ssVm6fB5kVePdv_I6dJl89YHnRjGnQmSIk5Uw-rDHGbycJvKxw
2014-07-13 14:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.45ms
2014-07-13 15:12:50 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 15:12:51 get_access_token.py[line:34] INFO access_token:DyO-WbcBGQ6JA6wJOuwIFZRnlAz3laRsH0idCEtCsdjvh19FcRrnC4gYAWaoS_peJuS_it0mByu7FwHX9kuGhQ
2014-07-13 15:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.43ms
2014-07-13 16:12:51 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 16:12:53 get_access_token.py[line:34] INFO access_token:ooXQUZgoqNCLap7VtMo_2nhr_vw7i1HjfJTvlBSxxpPx8SZW5OPCwywEZJ9Nr_5suscRur-frJxuO99VyKPcSg
2014-07-13 16:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.50ms
2014-07-13 17:12:53 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 17:12:55 get_access_token.py[line:34] INFO access_token:DhfWyy_uGF8t9nXxogUXeUTXbJR0WXCyl7KMvF0Y8ovI-8XOOSuGX-AO0I4Ks74S3aXC0sfCXAnGJJta1J0tyQ
2014-07-13 17:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.59ms
2014-07-13 18:12:55 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 18:12:56 get_access_token.py[line:34] INFO access_token:KCzwFqsFyIwQ7HYlx5OzfLAVz871EeMaaslu1ANpzKwmvOKKq_QuHooxOq2b6so7eZLcEOTOsSKOZsKoNnc4IA
2014-07-13 18:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.36ms
2014-07-13 19:12:57 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 19:12:58 get_access_token.py[line:34] INFO access_token:EKcFR1lQ3Kh26NnSPRIbufbk0WlbtmwfJhQv6_w73xGcMbx-tcbeCRwkXSzkBwiweMphVDDv-XusnKX5u9pBGw
2014-07-13 19:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.61ms
2014-07-13 20:12:58 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 20:13:00 get_access_token.py[line:34] INFO access_token:QbodCBOW9Wtq2dqjIaCSoC08HaDcPZwcaYOsgdVYqZRDSrWY_GYFnVmV51H78EW-c0u_3FONNfjUCuleaMLG8A
2014-07-13 20:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-13 21:13:00 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 21:13:01 get_access_token.py[line:34] INFO access_token:5ccWMym-x7A5aVZc9gC7MRz6LlOEIvr4PaYKQUVqHFUv4U7k4kpJpfrgB_5oETdsvb8eRegJpGo33ptR2EGUjA
2014-07-13 21:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.46ms
2014-07-13 22:13:02 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 22:13:03 get_access_token.py[line:34] INFO access_token:M53jiDZ5mMtw7ZiPogy-i7YntCdMYNc0u6PQi6i0Y4d6fppZ0pi6GXUXO4cjO15-E2oAeqmhHHkJNMM5mKu46A
2014-07-13 22:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.47ms
2014-07-13 23:13:03 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-13 23:13:05 get_access_token.py[line:34] INFO access_token:NsxXIYyDc39_pxAOa8-m9GUxOzTpF-NJIWvXJZ8h8ZkqAaF_3D_natB7Jgv6vJ69ymnBfheqxOmXtO4xcYMvCQ
2014-07-13 23:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-14 00:13:05 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 00:13:06 get_access_token.py[line:34] INFO access_token:JG_zOEJlcjrt-25rmgJaWhxiWb0Xkhc1bjxytbsNby81iEjoGdx7PXP-FBOpYq2vLy6iDER3LTRasRj5xAvCFQ
2014-07-14 00:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.37ms
2014-07-14 01:13:06 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 01:13:08 get_access_token.py[line:34] INFO access_token:hOcSVY-wu9dJjVgR-Pbvs9StQ919khJkz2sdKl6_HGqOqPxYVKoAc1WkSlZ6uVofvMctfeywfAiiklpIJNfMtA
2014-07-14 01:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.42ms
2014-07-14 02:13:08 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 02:13:10 get_access_token.py[line:34] INFO access_token:aYKMyfY_Uzn2U0G94DUmUGfC74DeYQkMqPAK6wRBUkDGJqUTsduGtOod04CRkc947WyjbCyWDcB6Vv6Pjd-y1Q
2014-07-14 02:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.44ms
2014-07-14 03:13:10 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 03:13:11 get_access_token.py[line:34] INFO access_token:fh06z9F4x9-oEWu24K7kDI0eW6-DeLumxtk8EVnpEXk-xBxt8xUrXF4C9Wk0abHCUEU13N15bv1HiyhAGwJcbA
2014-07-14 03:27:03 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.41ms
2014-07-14 04:13:11 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 04:13:13 get_access_token.py[line:34] INFO access_token:2VjtyEj9snKKyTSIvG0fkfVg2t08KJuQQNfgbGiVbPoUPbDS1SI0toMbGapkucEwSNjasfIcHDBdfCvnvm22TA
2014-07-14 04:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.54ms
2014-07-14 05:13:13 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 05:13:14 get_access_token.py[line:34] INFO access_token:7YtYD01ErTJEG4p6Fah7mbrjrfqz7JcyGYi2UlbNnyrUg6PY9oXjZ-2G-zEwz8BE9C2jkbWIxT89yc8jD-KciQ
2014-07-14 05:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.39ms
2014-07-14 06:13:15 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 06:13:16 get_access_token.py[line:34] INFO access_token:lEaSbOEUvOedV0EvsjdvuekSLlmk8P_Q23gl_VYFPDEIrgjQFZQI54HjrnjrTSABs9D4GXP9xGMKpgc0bWfyKw
2014-07-14 06:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-14 07:13:16 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 07:13:18 get_access_token.py[line:34] INFO access_token:QpEG7VnolcdIGOp8tZWW2LwGezXdr7prW8AQPKA5lCmZRDbR-wHBPAUzD7Jp65Y_0gQASOvTMB8dS09FHHiGXA
2014-07-14 07:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.62ms
2014-07-14 07:31:06 web.py[line:1728] INFO 200 POST /wechat?signature=b484e8199b15b601da8f0c211ae5ff9db3cf4074&timestamp=1405294265&nonce=197734439 (127.0.0.1) 1.28ms
2014-07-14 08:13:18 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 08:13:19 get_access_token.py[line:34] INFO access_token:DhfWyy_uGF8t9nXxogUXeTpDpusCKlwwMFKtYJ8yXSfo0hdyfOdoAVaChm60G2-T73Prd5aetFaWHCFw_JVGfw
2014-07-14 08:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.47ms
2014-07-14 09:13:19 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 09:13:21 get_access_token.py[line:34] INFO access_token:rVc8pgJoOWLOULUvRnSWKZfmp7iKEKalFXFlvBO_cCXqC61jlYltH4ZgbJm65c0IRFL6_jrITQMta0Ykz_H-GQ
2014-07-14 09:27:00 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.38ms
2014-07-14 10:13:10 web.py[line:1728] INFO 200 POST /wechat?signature=c4018ab09b8947f7fb98e43b14285c0b46eb63d2&timestamp=1405303990&nonce=1619446214 (127.0.0.1) 2.38ms
2014-07-14 10:13:21 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 10:13:22 get_access_token.py[line:34] INFO access_token:rVc8pgJoOWLOULUvRnSWKV4ozBZlX34yIeq50WXjT-SW9nj-URzKn_4a30UiOHeoDEqcmNbr4aHf-n9BwANtXw
2014-07-14 10:14:19 web.py[line:1728] INFO 200 POST /wechat?signature=6c87607b39015df82b047c6883208a5bd6d6c4d5&timestamp=1405304059&nonce=338611367 (127.0.0.1) 0.87ms
2014-07-14 10:14:49 web.py[line:1728] INFO 200 GET /bind/1000 (127.0.0.1) 7.34ms
2014-07-14 10:17:34 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:17:35 send_talk.py[line:82] INFO 47637 : 法国
2014-07-14 10:17:35 web.py[line:1728] INFO 200 POST /wechat?signature=cde4bdf12d9529d3781502bdb67159072165de9c&timestamp=1405304254&nonce=653868015 (127.0.0.1) 1031.34ms
2014-07-14 10:17:40 send_talk.py[line:64] INFO 请勿重复发送
2014-07-14 10:17:40 send_talk.py[line:82] INFO 47637 : 法国
2014-07-14 10:17:40 web.py[line:1728] INFO 200 POST /wechat?signature=42220271d409423b1988411a1a61daea448e1f06&timestamp=1405304260&nonce=1379137846 (127.0.0.1) 7.73ms
2014-07-14 10:17:56 send_talk.py[line:64] INFO 请勿发送密码
2014-07-14 10:17:56 send_talk.py[line:82] INFO 47637 : sxjzz7wIyel
2014-07-14 10:17:56 web.py[line:1728] INFO 200 POST /wechat?signature=48d93ba2fea990cfd40c0253a7d4e81814b7d86c&timestamp=1405304276&nonce=1896465889 (127.0.0.1) 3.08ms
2014-07-14 10:18:06 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=f18ea643306aa0cc2616a2f47090363e75a8d0f7&timestamp=1405304285&nonce=1406451552 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=f18ea643306aa0cc2616a2f47090363e75a8d0f7&timestamp=1405304285&nonce=1406451552', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 75, in post
    ret_render(Feedback)
UnboundLocalError: local variable 'Feedback' referenced before assignment
2014-07-14 10:18:06 web.py[line:1728] ERROR 500 POST /wechat?signature=f18ea643306aa0cc2616a2f47090363e75a8d0f7&timestamp=1405304285&nonce=1406451552 (127.0.0.1) 3.73ms
2014-07-14 10:20:53 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=9c92fd714596627d6c369c4edd24a4e9ccc8f470&timestamp=1405304452&nonce=447216478 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=9c92fd714596627d6c369c4edd24a4e9ccc8f470&timestamp=1405304452&nonce=447216478', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 75, in post
    logging.info(Feedback)
NameError: global name 'logging' is not defined
2014-07-14 10:20:53 web.py[line:1728] ERROR 500 POST /wechat?signature=9c92fd714596627d6c369c4edd24a4e9ccc8f470&timestamp=1405304452&nonce=447216478 (127.0.0.1) 12.81ms
2014-07-14 10:27:01 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.45ms
2014-07-14 10:27:19 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=6978f2ecfd7430beb8c1ec6844f30e1345524672&timestamp=1405304839&nonce=1474719390 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=6978f2ecfd7430beb8c1ec6844f30e1345524672&timestamp=1405304839&nonce=1474719390', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 24, in home
    r = requests.get(home_url, data=data, head=head)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/requests-2.2.1-py2.7.egg/requests/api.py", line 55, in get
    return request('get', url, **kwargs)
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/requests-2.2.1-py2.7.egg/requests/api.py", line 44, in request
    return session.request(method=method, url=url, **kwargs)
TypeError: request() got an unexpected keyword argument 'head'
2014-07-14 10:27:20 web.py[line:1728] ERROR 500 POST /wechat?signature=6978f2ecfd7430beb8c1ec6844f30e1345524672&timestamp=1405304839&nonce=1474719390 (127.0.0.1) 26.31ms
2014-07-14 10:28:53 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:28:53 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=09ea9bb893e9e890cdc098dd5bc4bb85c1f09380&timestamp=1405304932&nonce=728445679 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=09ea9bb893e9e890cdc098dd5bc4bb85c1f09380&timestamp=1405304932&nonce=728445679', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 27, in home
    home_original = json.loads(r.replace('=',':'),\
AttributeError: 'Response' object has no attribute 'replace'
2014-07-14 10:28:53 web.py[line:1728] ERROR 500 POST /wechat?signature=09ea9bb893e9e890cdc098dd5bc4bb85c1f09380&timestamp=1405304932&nonce=728445679 (127.0.0.1) 862.99ms
2014-07-14 10:29:38 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:29:38 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=1e7bdd6df9211e5e48549c0f436642c0d5b2ba32&timestamp=1405304977&nonce=1957237679 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=1e7bdd6df9211e5e48549c0f436642c0d5b2ba32&timestamp=1405304977&nonce=1957237679', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 28, in home
    encoding='GB18030')
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 365, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 383, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
2014-07-14 10:29:38 web.py[line:1728] ERROR 500 POST /wechat?signature=1e7bdd6df9211e5e48549c0f436642c0d5b2ba32&timestamp=1405304977&nonce=1957237679 (127.0.0.1) 922.77ms
2014-07-14 10:34:49 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:34:50 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=346766654644bfcf4d415dd919c42720a4b56321&timestamp=1405305289&nonce=2075668152 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=346766654644bfcf4d415dd919c42720a4b56321&timestamp=1405305289&nonce=2075668152', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 25, in home
    assert r.status_code==200
AssertionError
2014-07-14 10:34:50 web.py[line:1728] ERROR 500 POST /wechat?signature=346766654644bfcf4d415dd919c42720a4b56321&timestamp=1405305289&nonce=2075668152 (127.0.0.1) 903.40ms
2014-07-14 10:40:14 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:40:15 home.py[line:30] INFO 401
2014-07-14 10:40:15 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=f178b065d05ab3381c11bb97c0f31672b865f840&timestamp=1405305614&nonce=84806970 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=f178b065d05ab3381c11bb97c0f31672b865f840&timestamp=1405305614&nonce=84806970', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 33, in home
    encoding='GB18030')
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 365, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 383, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
2014-07-14 10:40:15 web.py[line:1728] ERROR 500 POST /wechat?signature=f178b065d05ab3381c11bb97c0f31672b865f840&timestamp=1405305614&nonce=84806970 (127.0.0.1) 872.58ms
2014-07-14 10:41:27 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:41:29 home.py[line:30] INFO 401
2014-07-14 10:41:29 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=aa0c5ea9b17708cfeee0e27ba0c524973dadf471&timestamp=1405305687&nonce=1371802709 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=aa0c5ea9b17708cfeee0e27ba0c524973dadf471&timestamp=1405305687&nonce=1371802709', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 32, in home
    home_original = json.loads(r.text.replace('=',':'), encoding='GB18030')
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 365, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 383, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
2014-07-14 10:41:29 web.py[line:1728] ERROR 500 POST /wechat?signature=aa0c5ea9b17708cfeee0e27ba0c524973dadf471&timestamp=1405305687&nonce=1371802709 (127.0.0.1) 1815.55ms
2014-07-14 10:43:47 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:43:48 home.py[line:30] INFO {u'wechat_id': u'oEnIQuD6AwgX4EDgPorkJx-tnvPM', u'_id': ObjectId('535241736d270c4dc51df569'), u'id': 172L}
2014-07-14 10:43:48 home.py[line:31] INFO http://weilairiji.com/api/statuses/user_timeline.json?id=172&page=1
2014-07-14 10:43:48 home.py[line:32] INFO 401
2014-07-14 10:43:48 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=2a05f78bef1dda36fd726748e3fb1a8890b25185&timestamp=1405305827&nonce=58300788 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=2a05f78bef1dda36fd726748e3fb1a8890b25185&timestamp=1405305827&nonce=58300788', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 34, in home
    home_original = json.loads(r.text.replace('=',':'), encoding='GB18030')
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 365, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 383, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
2014-07-14 10:43:48 web.py[line:1728] ERROR 500 POST /wechat?signature=2a05f78bef1dda36fd726748e3fb1a8890b25185&timestamp=1405305827&nonce=58300788 (127.0.0.1) 857.36ms
2014-07-14 10:45:05 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:45:06 home.py[line:30] INFO {u'_id': ObjectId('535241736d270c4dc51df569'), u'xiezhua_id': u'chenxinyiray@gmail.com:2162383cxy', u'id': 172L}
2014-07-14 10:45:06 home.py[line:31] INFO http://weilairiji.com/api/statuses/user_timeline.json?id=172&page=1
2014-07-14 10:45:06 home.py[line:32] INFO 401
2014-07-14 10:45:06 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=3db3c66ac516ffce5674f55d26543335e350539c&timestamp=1405305904&nonce=854867115 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=3db3c66ac516ffce5674f55d26543335e350539c&timestamp=1405305904&nonce=854867115', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 34, in home
    home_original = json.loads(r.text.replace('=',':'), encoding='GB18030')
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 365, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 383, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
2014-07-14 10:45:06 web.py[line:1728] ERROR 500 POST /wechat?signature=3db3c66ac516ffce5674f55d26543335e350539c&timestamp=1405305904&nonce=854867115 (127.0.0.1) 1873.18ms
2014-07-14 10:50:01 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:50:02 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=1465bf7e48798e94b2f93944a5afcf4303dfbba7&timestamp=1405306201&nonce=221491753 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=1465bf7e48798e94b2f93944a5afcf4303dfbba7&timestamp=1405306201&nonce=221491753', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 37, in home
    home_original))
  File "/home/nightwish/elfin/scripts/home.py", line 36, in <lambda>
    lambda item: item['text']+item['time']+'\n',\
KeyError: 'time'
2014-07-14 10:50:02 web.py[line:1728] ERROR 500 POST /wechat?signature=1465bf7e48798e94b2f93944a5afcf4303dfbba7&timestamp=1405306201&nonce=221491753 (127.0.0.1) 1579.03ms
2014-07-14 10:52:32 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:52:33 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=066ce33f69319f57e28ec6a0b5da55992e500089&timestamp=1405306352&nonce=2049389461 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=066ce33f69319f57e28ec6a0b5da55992e500089&timestamp=1405306352&nonce=2049389461', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 32, in home
    home_original))
TypeError: sum() can't sum strings [use ''.join(seq) instead]
2014-07-14 10:52:33 web.py[line:1728] ERROR 500 POST /wechat?signature=066ce33f69319f57e28ec6a0b5da55992e500089&timestamp=1405306352&nonce=2049389461 (127.0.0.1) 1459.36ms
2014-07-14 10:54:26 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:54:28 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=26843504834f486dd0ccd0c714916e347452ea15&timestamp=1405306466&nonce=1187044609 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=26843504834f486dd0ccd0c714916e347452ea15&timestamp=1405306466&nonce=1187044609', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 31, in home
    lambda item: item['text'], home_original))
TypeError: join() takes exactly one argument (2 given)
2014-07-14 10:54:28 web.py[line:1728] ERROR 500 POST /wechat?signature=26843504834f486dd0ccd0c714916e347452ea15&timestamp=1405306466&nonce=1187044609 (127.0.0.1) 1394.96ms
2014-07-14 10:55:42 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:55:45 web.py[line:1728] INFO 200 POST /wechat?signature=4d7edb8e312a40e23688e58fd413d21d56ba1828&timestamp=1405306542&nonce=943932810 (127.0.0.1) 2303.14ms
2014-07-14 10:55:45 web.py[line:1302] WARNING 400 GET /wechat (127.0.0.1): Missing argument signature
2014-07-14 10:55:45 web.py[line:1728] WARNING 400 GET /wechat (127.0.0.1) 1.35ms
2014-07-14 10:57:24 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:57:25 web.py[line:1728] INFO 200 POST /wechat?signature=5afdca29816828c62c98fd3b5e63a72f44dcaf37&timestamp=1405306644&nonce=2146533170 (127.0.0.1) 1435.48ms
2014-07-14 10:59:37 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 10:59:40 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 10:59:40 web.py[line:1728] INFO 200 POST /wechat?signature=308197b1fa5134210727b324e82325c65b8ea58d&timestamp=1405306777&nonce=2039032967 (127.0.0.1) 3383.63ms
2014-07-14 11:01:21 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:01:22 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 11:01:22 web.py[line:1728] INFO 200 POST /wechat?signature=5e035b21bb2012f28f656914fd68ea5bbc67ca7f&timestamp=1405306881&nonce=209929694 (127.0.0.1) 1441.59ms
2014-07-14 11:01:39 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:01:40 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 11:01:40 web.py[line:1728] INFO 200 POST /wechat?signature=6d3427b2ff4b71ae18bac3d8efc28123e438b38b&timestamp=1405306899&nonce=1584130491 (127.0.0.1) 918.22ms
2014-07-14 11:02:00 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:02:02 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 11:02:02 web.py[line:1728] INFO 200 POST /wechat?signature=63e599e3a08ebdf2468a69edccc7da46d858ffda&timestamp=1405306920&nonce=21100719 (127.0.0.1) 1971.93ms
2014-07-14 11:03:17 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:03:19 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 11:03:19 web.py[line:1728] INFO 200 POST /wechat?signature=21779828376da2a945444c6465b30cc0bdb222e9&timestamp=1405306997&nonce=1469712007 (127.0.0.1) 1980.36ms
2014-07-14 11:03:33 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:03:35 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 11:03:35 web.py[line:1728] INFO 200 POST /wechat?signature=3e618735731580c6fd0c1166907848381f1ca541&timestamp=1405307013&nonce=635007779 (127.0.0.1) 1152.91ms
2014-07-14 11:03:52 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:03:54 home.py[line:30] INFO [u'text', u'created_at', u'user', u'reply_to_user_id', u'id', u'reply_to_status_id']
2014-07-14 11:03:54 web.py[line:1728] INFO 200 POST /wechat?signature=aec33430f3774763eca7850202093f7e4428bbcb&timestamp=1405307032&nonce=522427557 (127.0.0.1) 1168.73ms
2014-07-14 11:08:13 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:08:14 home.py[line:32] INFO @紫绿苏蓝 310个字？

@蓝策° OK！ 现在蟹爪最大长度是三百，如果发送了350字的talk，是分成三百一条和五十一条，还是一百七十五个字各一条？

经过测试，最大容量竟然是三百个字！

@陆青琅。 好早以前就停掉了

@紫绿苏蓝 hello

尾巴成功更改为:相机

@蟹爪客服深幽 ^_^  会的  放心吧

@____Ｃ小姐。 没有艺术加工的真实人生经历

@蟹爪客服深幽 六四去广场寻衅滋事了

吸毒一点都不酷。毒瘾犯了的时候，他们全身发抖、冒汗流泪、幻听、激动、低吼、自虐、在地上翻滚。

RT蟹爪客服深幽 “冰妹”这个词第一次听说是在看守所里大毒枭的口中。

@renyi 三十一天整

【7】在天安门广场上把钱包翻开后有六个便衣从不同的方向向他扑来，他还托我出去后给他三十多岁的第三任妻子打个电话，我俩一样，都是在六四那天被秘密逮捕，被送往看守所后连向家里通知一声的手续都没有。他的徒孙遍天下，他在道上的外号叫“老大哥”----“Big Man&amp;quot;。

【6】铁夹子的用法等等。老爷子这次被抓是因为在天安门升国旗的时候扒窃了一个钱包，里面只有四十七块钱，法院判处有期徒刑六个月。算起来老爷子今年六十五岁，在监狱中度过的日子加起来也有二十多年了。他对我过下水道工业的历史很清楚，每个省的监狱他都呆过，包括香港、澳门、台湾。

【5】老爷子在看守所里跟我讲了“抠皮子”手艺的诸般练习法门。“抠皮子”最常用的手势是中指搭在食指上，搭上去后两指要长短一样，夹起来要有力、稳定。练习方法是在沸水或沸油中夹肥皂，水或油的温度、肥皂的大小形状都有很高的要求。也有借助器械的，法门更多，比如贴在牙龈上的刀片是特制的，

【4】去上海的火车上，哑巴师傅怕两人走散，就用一根布带把瘦弱的老爷子系在腰前，顺便还能胁下生风攒点上海过活的本钱。在上海，老爷子跟着哑巴师傅学会了这辈子吃饭的手艺，艺成之后挥师别去闯荡大江南北。他的哑巴师傅后来活到了一九八九年，无妻无子，收尸还靠的是徒子徒孙。

【3】成了他的师傅。老爷子说他的哑巴师傅能够从人前快速走过的同时解开老式中山装的胸前口袋、取钱出来、再把扣子系上而让人毫无察觉。如果说现今国内程序员竞争最激烈的地方是北京的话，那么二十世纪扒窃高手辈出风云际会的中心便是上海。徒孙二人后来趁着文革大串联坐火车来到了上海。

【2】老爷子说人生的转机出现在一次市集上，他看见有的人在挑菜还有个人在后面扒钱包，’扒窃‘黑话里叫‘抠皮子’，年幼的老爷子产生了恶作剧的想法，他拉了拉贼的衣服做手势要揭发，贼大惊，做了个噤声的手势，又比划了个吃饭的动作。后来贼真的请他下馆子吃了饭，意外的是，这贼是个哑巴，后来还

【1】和我关系最好的老爷子出生于新中国诞生的那一年，八岁赶上大跃进和人民公社化运动，江西的农村穷阿，人们吃不上饭，成片成片的死，路边有倒毙的死人和‘打嗝’的出生不久的婴儿，家里的兄弟姐妹也一个接一个地被裹进草席埋进土地。在那样的环境下活下来的不可能是好人，老爷子当年靠的是偷。

脾气最古怪的老爷子开了一辈子的车，直到有一天在路上遇到了碰瓷的老奶奶，老爷子下车和仰躺地上的老奶奶谈价钱，价钱没谈拢一个转身扣上车门，把车从老奶奶的身上开了过去，从此司机生涯终结了，他现在是看守所里年纪最大的杀人犯。
2014-07-14 11:08:14 web.py[line:1728] INFO 200 POST /wechat?signature=eb37d02e3157ca12a59aeba8dc912184c05d8f07&timestamp=1405307290&nonce=1874486419 (127.0.0.1) 1423.84ms
2014-07-14 11:10:40 web.py[line:1728] INFO 200 POST /wechat?signature=5be696c3561ff8a91306be2e97c185d9d677d29a&timestamp=1405307439&nonce=1303246372 (127.0.0.1) 7.83ms
2014-07-14 11:10:43 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 4.95ms
2014-07-14 11:10:43 web.py[line:1728] INFO 200 GET /static/jquery.min.js (127.0.0.1) 5.35ms
2014-07-14 11:10:44 web.py[line:1728] INFO 200 GET /static/base.css (127.0.0.1) 0.79ms
2014-07-14 11:10:45 web.py[line:1728] INFO 304 GET /static/logo.gif (127.0.0.1) 0.89ms
2014-07-14 11:11:15 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:11:16 login.py[line:38] ERROR No JSON object could be decoded
2014-07-14 11:11:16 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 860.17ms
2014-07-14 11:11:30 web.py[line:1728] INFO 200 POST /wechat?signature=cda9c92759787abb81a68f9835996de628701b39&timestamp=1405307489&nonce=1106544637 (127.0.0.1) 13.50ms
2014-07-14 11:13:22 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 11:13:24 get_access_token.py[line:34] INFO access_token:x_ZIZvbitNHTRLQG2XlWGiOHpPJvbCijOiCdna9mnvH8rLmu8rnvsxZHyCsFMRyo9q5jInJyLzrFdGFD7LVuOw
2014-07-14 11:15:09 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 7.12ms
2014-07-14 11:15:35 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:15:36 web.py[line:1305] ERROR Uncaught exception POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/bind/oDVlVuO4iES2MP1p83ilRdIIul1k', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Origin': 'http://bandwagon.shijunzi.info', 'Content-Length': '44', 'Accept-Language': 'zh-cn', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 6_1_3 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Mobile/10B329 MicroMessenger/5.3', 'Connection': 'close', 'Referer': 'http://bandwagon.shijunzi.info/elfin/bind/oDVlVuO4iES2MP1p83ilRdIIul1k', 'Content-Type': 'application/x-www-form-urlencoded'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 26, in post
    _login.analyses()
  File "/home/nightwish/elfin/scripts/login.py", line 40, in analyses
    user_info = ret_list['user']
TypeError: list indices must be integers, not str
2014-07-14 11:15:36 web.py[line:1728] ERROR 500 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 1104.44ms
2014-07-14 11:18:29 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:18:30 login.py[line:41] INFO [{u'text': u'\u5f02\u7a7a\u95f4\u4f20\u9001\u5668\u642d\u5efa\u5b8c\u6bd5\uff01', u'created_at': u'2014-07-14 11:07:09', u'user': {u'description': u'\u624b\u827a\u4eba\u800c\u5df2', u'url': u'http://weilairiji.com/home/u.24579', u'profile_image_url': u'http://weilairiji.com/attachments/head/24579.png?18735', u'protected': u'true', u'livelocation': u' ', u'homelocation': u' ', u'id': u'24579', u'name': u'\u6811\u4e0a\u7684\u7537\u7235'}, u'reply_to_user_id': u'', u'id': u'1071249179', u'reply_to_status_id': u''}]
2014-07-14 11:18:30 web.py[line:1305] ERROR Uncaught exception POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/bind/oDVlVuO4iES2MP1p83ilRdIIul1k', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Origin': 'http://bandwagon.shijunzi.info', 'Content-Length': '44', 'Accept-Language': 'zh-cn', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 6_1_3 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Mobile/10B329 MicroMessenger/5.3', 'Connection': 'close', 'Referer': 'http://bandwagon.shijunzi.info/elfin/bind/oDVlVuO4iES2MP1p83ilRdIIul1k', 'Content-Type': 'application/x-www-form-urlencoded'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 26, in post
    _login.analyses()
  File "/home/nightwish/elfin/scripts/login.py", line 42, in analyses
    user_info = ret_list['user']
TypeError: list indices must be integers, not str
2014-07-14 11:18:30 web.py[line:1728] ERROR 500 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 1056.33ms
2014-07-14 11:20:34 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:20:39 login.py[line:41] INFO [{u'text': u'\u5f02\u7a7a\u95f4\u4f20\u9001\u5668\u642d\u5efa\u5b8c\u6bd5\uff01', u'created_at': u'2014-07-14 11:07:18', u'user': {u'description': u'\u624b\u827a\u4eba\u800c\u5df2', u'url': u'http://weilairiji.com/home/u.24579', u'profile_image_url': u'http://weilairiji.com/attachments/head/24579.png?18735', u'protected': u'true', u'livelocation': u' ', u'homelocation': u' ', u'id': u'24579', u'name': u'\u6811\u4e0a\u7684\u7537\u7235'}, u'reply_to_user_id': u'', u'id': u'1071249180', u'reply_to_status_id': u''}]
2014-07-14 11:20:39 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 5057.22ms
2014-07-14 11:20:54 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:20:56 send_talk.py[line:82] INFO 1071249180 : 哈哈哈
2014-07-14 11:20:56 web.py[line:1728] INFO 200 POST /wechat?signature=311b06a6f7ca1199df350a684846331a8e5c22c4&timestamp=1405308053&nonce=1659897043 (127.0.0.1) 1883.42ms
2014-07-14 11:21:02 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:21:04 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=ad1b09b19928eaf4e7bc3943f9f3b22a70393383&timestamp=1405308062&nonce=548443122 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=ad1b09b19928eaf4e7bc3943f9f3b22a70393383&timestamp=1405308062&nonce=548443122', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '279', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 74, in post
    Feedback = home(fromUser, key)
  File "/home/nightwish/elfin/scripts/home.py", line 29, in home
    home_original = json.loads(r.text.replace('=',':'), encoding='GB18030')
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 365, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 383, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
2014-07-14 11:21:04 web.py[line:1728] ERROR 500 POST /wechat?signature=ad1b09b19928eaf4e7bc3943f9f3b22a70393383&timestamp=1405308062&nonce=548443122 (127.0.0.1) 1837.09ms
2014-07-14 11:27:02 web.py[line:1728] WARNING 404 GET / (23.252.104.24) 0.46ms
2014-07-14 11:30:30 web.py[line:1728] INFO 200 POST /wechat?signature=4f94021af898fe2664bb822631ab0ada0d690568&timestamp=1405308629&nonce=799955132 (127.0.0.1) 2.11ms
2014-07-14 11:30:52 web.py[line:1728] INFO 200 POST /wechat?signature=066245db5121781aba0d5dbdfa063929b8d95f93&timestamp=1405308652&nonce=747602864 (127.0.0.1) 5.67ms
2014-07-14 11:31:01 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 3.14ms
2014-07-14 11:31:03 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 0.62ms
2014-07-14 11:31:04 web.py[line:1728] INFO 200 GET /bind/static/jquery.min.js (127.0.0.1) 0.95ms
2014-07-14 11:31:31 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:31:32 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 1176.41ms
2014-07-14 11:31:48 web.py[line:1728] INFO 200 POST /wechat?signature=3f1c16a4fd93410e8f7d924db37adf9904df53d0&timestamp=1405308707&nonce=792749291 (127.0.0.1) 2.80ms
2014-07-14 11:33:26 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 0.87ms
2014-07-14 11:33:55 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:33:56 login.py[line:39] ERROR No JSON object could be decoded
2014-07-14 11:33:56 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 1844.20ms
2014-07-14 11:34:22 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:34:23 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 956.61ms
2014-07-14 11:34:32 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:34:34 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=5aedefd319bd2fdd27427300c2f600280c9e61a0&timestamp=1405308871&nonce=130214143 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=5aedefd319bd2fdd27427300c2f600280c9e61a0&timestamp=1405308871&nonce=130214143', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 50, in post
    ret_render(Feedback)
UnboundLocalError: local variable 'Feedback' referenced before assignment
2014-07-14 11:34:34 web.py[line:1728] ERROR 500 POST /wechat?signature=5aedefd319bd2fdd27427300c2f600280c9e61a0&timestamp=1405308871&nonce=130214143 (127.0.0.1) 1913.86ms
2014-07-14 11:36:11 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 7.02ms
2014-07-14 11:36:38 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:36:39 web.py[line:1305] ERROR Uncaught exception POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/bind/oDVlVuO4iES2MP1p83ilRdIIul1k', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Origin': 'http://bandwagon.shijunzi.info', 'Content-Length': '44', 'Accept-Language': 'zh-cn', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 6_1_3 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Mobile/10B329 MicroMessenger/5.3', 'Connection': 'close', 'Referer': 'http://bandwagon.shijunzi.info/elfin/bind/oDVlVuO4iES2MP1p83ilRdIIul1k', 'Content-Type': 'application/x-www-form-urlencoded'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 36, in post
    mongo.remove({'id':elfin['id']})
  File "build/bdist.linux-i686/egg/pymongo/collection.py", line 1677, in __call__
    self.__name)
TypeError: 'Collection' object is not callable. If you meant to call the 'remove' method on a 'Database' object it is failing because no such method exists.
2014-07-14 11:36:39 web.py[line:1728] ERROR 500 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 859.71ms
2014-07-14 11:42:42 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:42:43 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 1063.73ms
2014-07-14 11:42:56 web.py[line:1728] INFO 200 POST /wechat?signature=569d2a876bfd24ea8cc6c3b4a143043e4f9c2fd6&timestamp=1405309376&nonce=1672559038 (127.0.0.1) 8.20ms
2014-07-14 11:45:08 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 15.60ms
2014-07-14 11:45:36 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:45:40 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 3096.45ms
2014-07-14 11:45:49 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:45:50 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=9580f15d1196ef6f06b5bfc3cd139d747f8df69f&timestamp=1405309549&nonce=1866828794 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=9580f15d1196ef6f06b5bfc3cd139d747f8df69f&timestamp=1405309549&nonce=1866828794', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '277', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 50, in post
    ret_render(Feedback)
UnboundLocalError: local variable 'Feedback' referenced before assignment
2014-07-14 11:45:50 web.py[line:1728] ERROR 500 POST /wechat?signature=9580f15d1196ef6f06b5bfc3cd139d747f8df69f&timestamp=1405309549&nonce=1866828794 (127.0.0.1) 946.35ms
2014-07-14 11:58:38 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 11:58:39 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=1a8bf3c5602c600c463167e727484e4250a15036&timestamp=1405310318&nonce=1667194780 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=1a8bf3c5602c600c463167e727484e4250a15036&timestamp=1405310318&nonce=1667194780', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '278', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 49, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 82, in send
    logging.info(user['id'] + ' : ' + talk)
TypeError: unsupported operand type(s) for +: 'int' and 'str'
2014-07-14 11:58:39 web.py[line:1728] ERROR 500 POST /wechat?signature=1a8bf3c5602c600c463167e727484e4250a15036&timestamp=1405310318&nonce=1667194780 (127.0.0.1) 876.36ms
2014-07-14 12:03:09 web.py[line:1728] INFO 200 GET /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 7.71ms
2014-07-14 12:03:35 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:03:39 login.py[line:42] INFO {u'description': u'\u624b\u827a\u4eba\u800c\u5df2', u'url': u'http://weilairiji.com/home/u.24579', u'profile_image_url': u'http://weilairiji.com/attachments/head/24579.png?18735', u'protected': u'true', u'livelocation': u' ', u'homelocation': u' ', u'id': u'24579', u'name': u'\u6811\u4e0a\u7684\u7537\u7235'}
2014-07-14 12:03:39 web.py[line:1728] INFO 200 POST /bind/oDVlVuO4iES2MP1p83ilRdIIul1k (127.0.0.1) 3136.26ms
2014-07-14 12:05:11 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:05:12 web.py[line:1305] ERROR Uncaught exception POST /wechat?signature=2602c4217762c237a337b2094313691e87330247&timestamp=1405310711&nonce=1181282584 (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/wechat?signature=2602c4217762c237a337b2094313691e87330247&timestamp=1405310711&nonce=1181282584', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Content-Length': '276', 'Host': '127.0.0.1:9001', 'Pragma': 'no-cache', 'Content-Type': 'text/xml', 'Connection': 'close', 'Accept': '*/*', 'User-Agent': 'Mozilla/4.0'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/wechat.py", line 49, in post
    Feedback = send(fromUser, Text)
  File "/home/nightwish/elfin/scripts/send_talk.py", line 82, in send
    logging.info(user['id'] + ' : ' + talk)
TypeError: unsupported operand type(s) for +: 'int' and 'str'
2014-07-14 12:05:12 web.py[line:1728] ERROR 500 POST /wechat?signature=2602c4217762c237a337b2094313691e87330247&timestamp=1405310711&nonce=1181282584 (127.0.0.1) 591.41ms
2014-07-14 12:06:25 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:06:25 send_talk.py[line:82] INFO 24579 : Shock
2014-07-14 12:06:25 web.py[line:1728] INFO 200 POST /wechat?signature=cecbdf42f03aa9fddf0a946b09f665f59bb25d79&timestamp=1405310784&nonce=424423591 (127.0.0.1) 851.54ms
2014-07-14 12:06:35 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:06:35 send_talk.py[line:82] INFO 24579 : 觉得
2014-07-14 12:06:35 web.py[line:1728] INFO 200 POST /wechat?signature=823831e434a0fe62fd5acda519e85b0a6d32460c&timestamp=1405310794&nonce=484904786 (127.0.0.1) 605.42ms
2014-07-14 12:06:42 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:06:44 web.py[line:1728] INFO 200 POST /wechat?signature=46284510deeded0c3fbed9b39ea0c076514077b7&timestamp=1405310802&nonce=1803855990 (127.0.0.1) 1382.80ms
2014-07-14 12:06:56 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:06:57 web.py[line:1728] INFO 200 POST /wechat?signature=9af90a47f7386f34c371732cd01fe21d3c232c76&timestamp=1405310816&nonce=1726713084 (127.0.0.1) 1100.18ms
2014-07-14 12:07:03 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:07:04 web.py[line:1728] INFO 200 POST /wechat?signature=4e8d1d35fad6999edad97a766690b71a858e0e09&timestamp=1405310823&nonce=234501666 (127.0.0.1) 878.94ms
2014-07-14 12:07:38 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:07:39 web.py[line:1728] INFO 200 POST /wechat?signature=44758c4492b4fe1d882afd2ff75623ad3b87accc&timestamp=1405310858&nonce=2143937976 (127.0.0.1) 901.34ms
2014-07-14 12:07:51 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:07:53 send_talk.py[line:82] INFO 47637 : 哈哈哈
2014-07-14 12:07:53 web.py[line:1728] INFO 200 POST /wechat?signature=2faab8f18ee4acb281f1bd089ad909603bc524c9&timestamp=1405310871&nonce=1250694412 (127.0.0.1) 1979.82ms
2014-07-14 12:13:24 connectionpool.py[line:635] INFO Starting new HTTPS connection (1): api.weixin.qq.com
2014-07-14 12:13:26 get_access_token.py[line:34] INFO access_token:kRnek6Gb1JPpniDmVsSCiPwC6ORPAWqTc52bSrIlfLtor7zCYkqMvOiNvRWOMFX1esMgYxAUyzMGMlNJM_uZpQ
2014-07-14 12:17:51 web.py[line:1728] INFO 200 POST /wechat?signature=15d0bd098a10634f4b69d79404b1dcc1e6e44b38&timestamp=1405311471&nonce=669145903 (127.0.0.1) 2.04ms
2014-07-14 12:18:09 web.py[line:1728] INFO 200 POST /wechat?signature=f645dd0e929bb036fd526584d26a9e6c7704b107&timestamp=1405311487&nonce=1367812497 (127.0.0.1) 5.56ms
2014-07-14 12:18:16 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 4.30ms
2014-07-14 12:18:44 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:18:45 login.py[line:42] INFO {u'description': u'\u8bc5\u5492\u516c\u68c0\u6cd5', u'url': u'http://weilairiji.com/home/u.47637', u'profile_image_url': u'http://weilairiji.com/attachments/head/47637.jpg?20680', u'protected': u'false', u'livelocation': u'\u5317\u4eac \u5317\u4eac', u'homelocation': u'\u6cb3\u5357\u7701 \u9a7b\u9a6c\u5e97', u'id': u'47637', u'name': u'\u96f7\u950b'}
2014-07-14 12:18:45 web.py[line:1305] ERROR Uncaught exception POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1)
HTTPRequest(protocol='http', host='127.0.0.1:9001', method='POST', uri='/bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM', version='HTTP/1.0', remote_ip='127.0.0.1', headers={'Origin': 'http://bandwagon.shijunzi.info', 'Content-Length': '44', 'Accept-Language': 'zh-cn', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:9001', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 6_1_3 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Mobile/10B329 MicroMessenger/5.3', 'Connection': 'close', 'Referer': 'http://bandwagon.shijunzi.info/elfin/bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM', 'Content-Type': 'application/x-www-form-urlencoded'})
Traceback (most recent call last):
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1218, in _when_complete
    callback()
  File "/home/nightwish/elfin/local/lib/python2.7/site-packages/tornado-3.2-py2.7-linux-i686.egg/tornado/web.py", line 1239, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/nightwish/elfin/handlers/bind.py", line 38, in post
    time=time.ctime(), action=action)
UnboundLocalError: local variable 'action' referenced before assignment
2014-07-14 12:18:45 web.py[line:1728] ERROR 500 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1137.95ms
2014-07-14 12:20:12 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:20:13 login.py[line:42] INFO {u'description': u'\u8bc5\u5492\u516c\u68c0\u6cd5', u'url': u'http://weilairiji.com/home/u.47637', u'profile_image_url': u'http://weilairiji.com/attachments/head/47637.jpg?20680', u'protected': u'false', u'livelocation': u'\u5317\u4eac \u5317\u4eac', u'homelocation': u'\u6cb3\u5357\u7701 \u9a7b\u9a6c\u5e97', u'id': u'47637', u'name': u'\u96f7\u950b'}
2014-07-14 12:20:13 web.py[line:1728] INFO 200 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 1347.34ms
2014-07-14 12:20:25 web.py[line:1728] INFO 200 POST /wechat?signature=3f5da7b29ac2ce0b5918a151f091d989fc7a29c2&timestamp=1405311625&nonce=1218992549 (127.0.0.1) 5.09ms
2014-07-14 12:20:27 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.87ms
2014-07-14 12:21:49 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.84ms
2014-07-14 12:21:49 web.py[line:1728] INFO 200 GET /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 0.97ms
2014-07-14 12:21:49 web.py[line:1728] INFO 200 GET /bind/static/jquery.min.js (127.0.0.1) 0.74ms
2014-07-14 12:22:20 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:22:21 login.py[line:39] ERROR No JSON object could be decoded
2014-07-14 12:22:21 web.py[line:1728] INFO 200 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 931.23ms
2014-07-14 12:22:53 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:22:54 login.py[line:42] INFO {u'description': u'\u8bc5\u5492\u516c\u68c0\u6cd5', u'url': u'http://weilairiji.com/home/u.47637', u'profile_image_url': u'http://weilairiji.com/attachments/head/47637.jpg?20680', u'protected': u'false', u'livelocation': u'\u5317\u4eac \u5317\u4eac', u'homelocation': u'\u6cb3\u5357\u7701 \u9a7b\u9a6c\u5e97', u'id': u'47637', u'name': u'\u96f7\u950b'}
2014-07-14 12:22:54 web.py[line:1728] INFO 200 POST /bind/oDVlVuGK_0DLJlDVxRsAIbHtD-CM (127.0.0.1) 618.65ms
2014-07-14 12:23:23 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:23:24 web.py[line:1728] INFO 200 POST /wechat?signature=76c5ea4d56a8ba4f8815583f5c65d248b0eed48e&timestamp=1405311803&nonce=1693616284 (127.0.0.1) 1233.39ms
2014-07-14 12:23:40 connectionpool.py[line:171] INFO Starting new HTTP connection (1): weilairiji.com
2014-07-14 12:23:41 send_talk.py[line:82] INFO 47637 : Hashed
2014-07-14 12:23:41 web.py[line:1728] INFO 200 POST /wechat?signature=9c86850944913544481d0a4c23db4b259b9b7fe2&timestamp=1405311819&nonce=1949006132 (127.0.0.1) 982.67ms
